<!doctype html>

<html lang="en-US" class="no-js">
  <head>
    <meta charset="utf-8">

<!-- begin _includes/seo.html --><title>A tutorial on Genome-Wide Association Studies (GWAS) in Tassel (GUI) | M. Shamshad</title>
<meta name="description" content="Genome-wide association studies (GWAS) increase their popularity among medical, biological, and social sciences to identify the association between single nucleotide polymorphisms and phenotypic traits. This tutorial aims to provide a guidelines for conducing genome wide analysis in Tassel.">


  <meta name="author" content="M. Shamshad">
  
  <meta property="article:author" content="M. Shamshad">
  


<meta property="og:type" content="article">
<meta property="og:locale" content="en_US">
<meta property="og:site_name" content="M. Shamshad">
<meta property="og:title" content="A tutorial on Genome-Wide Association Studies (GWAS) in Tassel (GUI)">
<meta property="og:url" content="http://localhost:4000/posts/2021/01/A-tutorial-on-Genome-Wide-Association-Studies/">


  <meta property="og:description" content="Genome-wide association studies (GWAS) increase their popularity among medical, biological, and social sciences to identify the association between single nucleotide polymorphisms and phenotypic traits. This tutorial aims to provide a guidelines for conducing genome wide analysis in Tassel.">



  <meta property="og:image" content="http://localhost:4000/images/biostat/gwas/gwaslogo.webp">



  <meta name="twitter:site" content="@DrShamshadM">
  <meta name="twitter:title" content="A tutorial on Genome-Wide Association Studies (GWAS) in Tassel (GUI)">
  <meta name="twitter:description" content="Genome-wide association studies (GWAS) increase their popularity among medical, biological, and social sciences to identify the association between single nucleotide polymorphisms and phenotypic traits. This tutorial aims to provide a guidelines for conducing genome wide analysis in Tassel.">
  <meta name="twitter:url" content="http://localhost:4000/posts/2021/01/A-tutorial-on-Genome-Wide-Association-Studies/">

  
    <meta name="twitter:card" content="summary_large_image">
    <meta name="twitter:image" content="http://localhost:4000/images/biostat/gwas/gwaslogo.webp">
  

  
    <meta name="twitter:creator" content="@DrShamshadM">
  



  <meta property="article:published_time" content="2021-01-20T00:00:00+05:30">



  <meta property="article:modified_time" content="2024-10-06T20:53:46+05:30">



  

  


<link rel="canonical" href="http://localhost:4000/posts/2021/01/A-tutorial-on-Genome-Wide-Association-Studies/">












<!-- end _includes/seo.html -->


  <link href="/feed.xml" type="application/atom+xml" rel="alternate" title="M. Shamshad Feed">


<!-- https://web.dev/viewport/?utm_source=lighthouse&utm_medium=lr --><meta name="viewport" content="width=device-width, initial-scale=1.0">

<script defer>
  document.documentElement.className = document.documentElement.className.replace(/\bno-js\b/g, '') + ' js ';
  window.enable_copy_code_button = true;
</script>
<link rel="stylesheet" href="/assets/academicons-1.9.4/css/academicons.min.css" fetchpriority="low"/>
<!-- For all browsers -->
<link
  rel="stylesheet"
  href="/assets/css/main.css"
  id="theme_source"
/>


<link
  rel="stylesheet alternate"
  href="/assets/css/mainDark.css"
  id="theme_source_2"
/>
<script defer>
  let theme = sessionStorage.getItem("theme");
  if (theme === "dark" || window.matchMedia('(prefers-color-scheme: dark)').matches) {
    sessionStorage.setItem("theme", "dark");
    node1 = document.getElementById("theme_source");
    node2 = document.getElementById("theme_source_2");
    node1.setAttribute("rel", "stylesheet alternate");
    node2.setAttribute("rel", "stylesheet");
  } else {
    sessionStorage.setItem("theme", "light");
  }
</script>


    
<!-- start custom head snippets -->			   
<meta name="apple-mobile-web-app-capable" content="yes" />
<meta name="apple-mobile-web-app-status-bar-style" content="default" />
<link rel="apple-touch-icon" sizes="60x60" href="/images/favicon/apple-touch-icon-60x60.png?v=565802">
<link rel="apple-touch-icon" sizes="76x76" href="/images/favicon/apple-touch-icon-76x76.png?v=565802">
<link rel="apple-touch-icon" sizes="120x120" href="/images/favicon/apple-touch-icon-120x120.png?v=565802">
<link rel="apple-touch-icon" sizes="152x152" href="/images/favicon/apple-touch-icon-152x152.png?v=565802">
<link rel="apple-touch-icon" sizes="180x180" href="/images/favicon/apple-touch-icon-180x180.png?v=565802">
<link rel="icon" type="image/png" sizes="32x32" href="/images/favicon/favicon-32x32.png?v=565802">
<link rel="icon" type="image/png" sizes="16x16" href="/images/favicon/favicon-16x16.png?v=565802">
<link rel="manifest" href="/images/favicon/site.webmanifest?v=565802">
<link rel="mask-icon" href="/images/favicon/safari-pinned-tab.svg?v=565802" color="#00a7ff">
<link rel="shortcut icon" href="/images/favicon/favicon.ico?v=565802">
<meta name="apple-mobile-web-app-title" content="SM-Web">
<meta name="application-name" content="SM-Web">
<meta name="msapplication-TileColor" content="#00a300">
<meta name="msapplication-config" content="/images/favicon/browserconfig.xml?v=565802">
<meta name="theme-color" content="#ffffff">

<!-- SEO tags -->
<!-- Begin Jekyll SEO tag v2.8.0 -->
<title>A tutorial on Genome-Wide Association Studies (GWAS) in Tassel (GUI) | M. Shamshad</title>
<meta name="generator" content="Jekyll v3.10.0" />
<meta property="og:title" content="A tutorial on Genome-Wide Association Studies (GWAS) in Tassel (GUI)" />
<meta name="author" content="M. Shamshad" />
<meta property="og:locale" content="en_US" />
<meta name="description" content="Genome-wide association studies (GWAS) increase their popularity among medical, biological, and social sciences to identify the association between single nucleotide polymorphisms and phenotypic traits. This tutorial aims to provide a guidelines for conducing genome wide analysis in Tassel." />
<meta property="og:description" content="Genome-wide association studies (GWAS) increase their popularity among medical, biological, and social sciences to identify the association between single nucleotide polymorphisms and phenotypic traits. This tutorial aims to provide a guidelines for conducing genome wide analysis in Tassel." />
<link rel="canonical" href="http://localhost:4000/posts/2021/01/A-tutorial-on-Genome-Wide-Association-Studies/" />
<meta property="og:url" content="http://localhost:4000/posts/2021/01/A-tutorial-on-Genome-Wide-Association-Studies/" />
<meta property="og:site_name" content="M. Shamshad" />
<meta property="og:type" content="article" />
<meta property="article:published_time" content="2021-01-20T00:00:00+05:30" />
<meta name="twitter:card" content="summary" />
<meta property="twitter:title" content="A tutorial on Genome-Wide Association Studies (GWAS) in Tassel (GUI)" />
<meta name="twitter:site" content="@DrShamshadM" />
<meta name="twitter:creator" content="@DrShamshadM" />
<script type="application/ld+json">
{"@context":"https://schema.org","@type":"BlogPosting","author":{"@type":"Person","name":"M. Shamshad"},"dateModified":"2024-10-06T20:53:46+05:30","datePublished":"2021-01-20T00:00:00+05:30","description":"Genome-wide association studies (GWAS) increase their popularity among medical, biological, and social sciences to identify the association between single nucleotide polymorphisms and phenotypic traits. This tutorial aims to provide a guidelines for conducing genome wide analysis in Tassel.","headline":"A tutorial on Genome-Wide Association Studies (GWAS) in Tassel (GUI)","mainEntityOfPage":{"@type":"WebPage","@id":"http://localhost:4000/posts/2021/01/A-tutorial-on-Genome-Wide-Association-Studies/"},"publisher":{"@type":"Organization","logo":{"@type":"ImageObject","url":"http://localhost:4000/images/favicon/site-treaser.webp"},"name":"M. Shamshad"},"url":"http://localhost:4000/posts/2021/01/A-tutorial-on-Genome-Wide-Association-Studies/"}</script>
<!-- End Jekyll SEO tag -->

  </head>

  <body class="layout--single" dir="ltr">
    <nav class="skip-links">
  <ul>
    <li><a href="#site-nav" class="screen-reader-shortcut">Skip to primary navigation</a></li>
    <li><a href="#main" class="screen-reader-shortcut">Skip to content</a></li>
    <li><a href="#footer" class="screen-reader-shortcut">Skip to footer</a></li>
  </ul>
</nav>

    

<div class="masthead">
  <div class="masthead__inner-wrap">
    <div class="masthead__menu">
      <nav id="site-nav" class="greedy-nav">
        
          <a class="site-logo" href="/"><img src="/images/favicon/site-treaser.webp" alt="M. Shamshad"></a>
        
        <a class="site-title" href="/">
          M. Shamshad
          
        </a>
        <ul class="visible-links">
<li class="masthead__menu-item">
              <a href="/about/">About</a>
            </li>
<li class="masthead__menu-item">
              <a href="/teaching/">Teaching</a>
            </li>
<li class="masthead__menu-item">
              <a href="/publications/">Publications</a>
            </li>
<li class="masthead__menu-item">
              <a href="/talks/">Talks</a>
            </li>
<li class="masthead__menu-item">
              <a href="/research/">Research</a>
            </li>
<li class="masthead__menu-item">
              <a href="/resume/">Resume</a>
            </li>
<li class="masthead__menu-item">
              <a href="/tags/">Tags</a>
            </li>
<li class="masthead__menu-item">
              <a href="/docs/Linux_tutorial/">Bioinfo</a>
            </li>
</ul>

        <span class="fa-solid fa-moon toggle-switch" type="button" style="cursor: pointer;" aria-hidden="true" onclick="node1 = document.getElementById('theme_source');
                 node2 = document.getElementById('theme_source_2');
        if(node1.getAttribute('rel')=='stylesheet'){node1.setAttribute('rel', 'stylesheet alternate'); 
            node2.setAttribute('rel', 'stylesheet');
            sessionStorage.setItem('theme', 'dark');}
            else{node2.setAttribute('rel', 'stylesheet alternate');
            node1.setAttribute('rel', 'stylesheet');sessionStorage.setItem('theme', 'light');} 
            return false;"></span>

        
        
        <button class="greedy-nav__toggle hidden" type="button">
          <span class="visually-hidden">Toggle menu</span>
          <div class="navicon"></div>
        </button>
        <ul class="hidden-links hidden"></ul>
      </nav>
    </div>
  </div>
</div>


    <div class="initial-content">
      





<div id="main" role="main">
  
  <div class="sidebar sticky">
  




<div itemscope itemtype="https://schema.org/Person" class="h-card">

  
  <div class="author__avatar">
   <a href="http://localhost:4000/">
    	<img loading="lazy" src="/images/profile.webp" alt="M. Shamshad" itemprop="image" class="u-photo">
   </a>
  </div>
  

  <div class="author__content">
    <h3 class="author__name p-name" itemprop="name">
      <a class="u-url" rel="me" href="http://localhost:4000/" itemprop="url">M. Shamshad</a>
    </h3>
    
      <div class="author__bio p-note" itemprop="description">
        <p>Plant Breeding &amp;&amp; Data Science</p>

      </div>
    
  </div>

  <div class="author__urls-wrapper">
    <button class="btn btn--inverse">Follow</button>
    <ul class="author__urls social-icons">
      
        <li itemprop="homeLocation" itemscope itemtype="https://schema.org/Place">
          <i class="fas fa-fw fa-map-marker-alt" aria-hidden="true"></i> <span itemprop="name" class="p-locality">Ludhina, Punjab <img class="emoji" title=":india:" alt=":india:" src="https://github.githubassets.com/images/icons/emoji/unicode/1f1ee-1f1f3.png" height="20" width="20"></span>
        </li>
      

      

      

      
        <li>
          <a href="mailto:shamshad.rattan@gmail.com" rel="me" class="u-email">
            <meta itemprop="email" content="shamshad.rattan@gmail.com">
            <i class="fas fa-fw fa-envelope" aria-hidden="true"></i><span class="label">Email</span>
          </a>
        </li>
      

      
       
        <li>
<a href="https://www.researchgate.net/profile/Mohd-Shamshad" itemprop="sameAs" rel="nofollow noopener noreferrer me">
          <i class="ai  ai-researchgate-square" aria-hidden="true"></i><span class="label">ResearchGate</span>
        </a>
      </li>
      

      
        <li>
          <a href="https://twitter.com/DrShamshadM" itemprop="sameAs" rel="nofollow noopener noreferrer me">
            <i class="fab fa-fw fa-twitter-square" aria-hidden="true"></i><span class="label">Twitter</span>
          </a>
        </li>
      

      

      
        <li>
          <a href="https://www.linkedin.com/in/dr-m-shamshad-58b4321b1" itemprop="sameAs" rel="nofollow noopener noreferrer me">
            <i class="fab fa-fw fa-linkedin" aria-hidden="true"></i><span class="label">LinkedIn</span>
          </a>
        </li>
      

      

      

      

      

      

      

      

      

      

      

      

      

      

      

      

      

      

       
      
      <li><a href="https://scholar.google.com/citations?user=dI7V1a4AAAAJ&amp;hl=en" itemprop="sameAs" rel="nofollow noopener noreferrer me">
        <i class="ai ai-fw ai-google-scholar-square" aria-hidden="true"></i><span class="label">Google Scholar</span></a></li>
                 
      
      
                                                                                                   
       
    </ul>
  </div>
</div>

  
  </div>


  <article class="page" itemscope itemtype="https://schema.org/CreativeWork">
    <meta itemprop="headline" content="A tutorial on Genome-Wide Association Studies (GWAS) in Tassel (GUI)">
    <meta itemprop="description" content="Genome-wide association studies (GWAS) increase their popularity among medical, biological, and social sciences to identify the association between single nucleotide polymorphisms and phenotypic traits. This tutorial aims to provide a guidelines for conducing genome wide analysis in Tassel.">
    <meta itemprop="datePublished" content="2021-01-20T00:00:00+05:30">
    <meta itemprop="dateModified" content="2024-10-06T20:53:46+05:30">

    <div class="page__inner-wrap">
      
        <header>
          <h1 id="page-title" class="page__title" itemprop="headline">
            <a href="http://localhost:4000/posts/2021/01/A-tutorial-on-Genome-Wide-Association-Studies/" itemprop="url">A tutorial on Genome-Wide Association Studies (GWAS) in Tassel (GUI)
</a>
          </h1>	      
          

  <p class="page__meta">
    

    

    
      
      

      <span class="page__meta-readtime">
        <i class="far fa-clock" aria-hidden="true"></i>
        
          3 minute read
        
      </span>
    
  </p>

   
        
          <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Updated:</strong> <time datetime="2021-01-20T00:00:00+05:30">January 20, 2021</time></p>
          
       
        </header>
      

      <section class="page__content" itemprop="text">
        
          <aside class="sidebar__right ">
            <nav class="toc">
              <header><h4 class="nav__title">
<i class="fas fa-file-alt"></i> On this page</h4></header>
              <ul class="toc__menu">
<li><a href="#tassel">TASSEL</a></li>
<li>
<a href="#10-download-and-install-tassel-software">1.0 Download and install TASSEL software</a><ul>
<li><a href="#11-preparing-the-input-files">1.1: Preparing the Input files</a></li>
<li><a href="#12-importing-phenotype-and-genotype-files">1.2: Importing phenotype and genotype files</a></li>
<li><a href="#13-checking-data-on-the-basis-of-phenotype-distribution-plot">1.3 Checking data on the basis of Phenotype distribution plot</a></li>
<li><a href="#15-genotype-summary-analysis">1.5 Genotype summary analysis</a></li>
</ul>
</li>
<li>
<a href="#20-conduct-gwas-analysis">2.0 Conduct GWAS analysis</a><ul>
<li><a href="#21-multidimensional-scaling-mds">2.1 multidimensional scaling (MDS)</a></li>
<li><a href="#22-intersecting-the-files">2.2 Intersecting the files</a></li>
</ul>
</li>
<li><a href="#30-running-general-linear-model-glm">3.0 running General Linear Model (GLM)</a></li>
<li>
<a href="#40-mixed-linear-model-mlm">4.0 Mixed Linear Model (MLM)</a><ul>
<li><a href="#41-calculating-kinship-matrix">4.1 Calculating Kinship matrix</a></li>
<li><a href="#42-running-mixed-linear-model-mlm">4.2 running Mixed Linear Model (MLM)</a></li>
<li><a href="#42-exporting-results">4.2 Exporting results</a></li>
<li><a href="#43-significance-threshold">4.3 Significance Threshold</a></li>
</ul>
</li>
</ul>
            </nav>
          </aside>
        
        <p>Genome-wide association studies (GWAS) increase their popularity among medical, biological, and social sciences to identify the association between single nucleotide polymorphisms and phenotypic traits. This tutorial aims to provide a guidelines for conducing genome wide analysis in Tassel.</p>

<!--more-->

<h2 id="tassel">TASSEL</h2>
<p><strong>TASSEL</strong> also known as for the <code class="language-plaintext highlighter-rouge">Evaluate traits aSSociations, Evolutionary Patterns, and linkage disequilibrium.</code> It is a powerful statistical software to conduct <code class="language-plaintext highlighter-rouge">association mapping</code> such as <code class="language-plaintext highlighter-rouge">General Linear Model (GLM)</code> and <code class="language-plaintext highlighter-rouge">Mixed Linear Model (MLM)</code>. The Tassel has ability to handle a wide range of indels (insertion &amp; deletions).</p>

<p>The GUI (graphical user interface) version of TASSEL is very well built for anyone who does not have a background or experience in working in <code class="language-plaintext highlighter-rouge">command line</code>. I will show how to prepare <code class="language-plaintext highlighter-rouge">input</code> files and run association analysis in TASSEL. For detailed information on TASSEL, user’s guide and further documentation please visit: 
<a href="https://www.maizegenetics.net/tassel">https://www.maizegenetics.net/tassel</a></p>

<h2 id="10-download-and-install-tassel-software">1.0 Download and install TASSEL software</h2>
<p>Download and install the latest version of the <strong> TASSEL software </strong> at this link:
<a href="https://www.maizegenetics.net/tassel">https://www.maizegenetics.net/tassel</a></p>

<hr>

<h3 id="11-preparing-the-input-files">1.1: Preparing the Input files</h3>

<p><b><u>Phenotype  file</u></b></p>

<p>Prepare the phenotype file as shown below in the figure, and please remember if your data has <code class="language-plaintext highlighter-rouge">covariates</code> such as <code class="language-plaintext highlighter-rouge">sex</code>, <code class="language-plaintext highlighter-rouge">age</code> or <code class="language-plaintext highlighter-rouge">treatment</code>, then, please categories them with header name <code class="language-plaintext highlighter-rouge">factor</code>.</p>

<center><img src="/images/biostat/gwas/pheno.png" alt="Phenotype data"></center>

<hr>
<p><b><u> Genotype file</u></b></p>

<p>TASSEL allows various genotype file formats such as <code class="language-plaintext highlighter-rouge">VCF</code> (variant call format), <code class="language-plaintext highlighter-rouge">.hmp.txt</code>, and <code class="language-plaintext highlighter-rouge">plink</code>. In this tutorial, I am using the <code class="language-plaintext highlighter-rouge">hmp.txt</code> version of the genotype file. The below githe screenshot of the hmp.txt genotype file.</p>

<center><img src="/images/biostat/gwas/geno.JPG" alt="Genotype data"></center>

<hr>

<h3 id="12-importing-phenotype-and-genotype-files">1.2: Importing phenotype and genotype files</h3>
<p>Import the files by following the steps shown below.</p>
<blockquote>
  <p><strong>Tip!</strong> Both files can be opened at same time holding <code class="language-plaintext highlighter-rouge">CTRL</code> and clicking the file names.</p>
</blockquote>

<center><img src="/images/biostat/gwas/importfiles.gif" alt="Import data"></center>

<hr>

<h3 id="13-checking-data-on-the-basis-of-phenotype-distribution-plot">1.3 Checking data on the basis of Phenotype distribution plot</h3>
<p>It is always a wise idea to look at the phenotype distribution by plotting to check for any outliers. Follow below steps to plot histogram of your phenotype data.</p>

<center><img src="/images/biostat/gwas/phenoplot.gif" alt="Phenotype distribution"></center>

<hr>

<h3 id="15-genotype-summary-analysis">1.5 Genotype summary analysis</h3>
<p>Next crucial step is to look at the genotype data by simply following the steps shown.
Couple of keys things to look at are:</p>
<ol>
  <li>Minor allele frequency distribution </li>
  <li>Missing genotypic data to see if it requires to be imputed </li>
  <li>Proportion of heterozygous in the samples to check for self-ed samples </li>
</ol>

<center><img src="/images/biostat/gwas/genosummary.gif" alt="Genotype summary"></center>
<hr>

<h2 id="20-conduct-gwas-analysis">2.0 Conduct GWAS analysis</h2>

<h3 id="21-multidimensional-scaling-mds">2.1 multidimensional scaling (MDS)</h3>
<p>MDS output can be used as the covariate in the GLM or MLM to correct for population structure.
Please follow the steps shown below:</p>

<center><img src="/images/biostat/gwas/mds.gif" alt="MDS"></center>

<hr>

<h3 id="22-intersecting-the-files">2.2 Intersecting the files</h3>
<p>Intersect the <code class="language-plaintext highlighter-rouge">genotype</code>, <code class="language-plaintext highlighter-rouge">phenotype</code> and <code class="language-plaintext highlighter-rouge">MDS</code> files by following the steps below:</p>

<center><img src="/images/biostat/gwas/intersect.gif" alt="Intersect files"></center>

<hr>

<h2 id="30-running-general-linear-model-glm">3.0 running General Linear Model (GLM)</h2>
<p>Run the GLM analysis by selecting the <code class="language-plaintext highlighter-rouge">intersected</code> files following the steps below:</p>

<center><img src="/images/biostat/gwas/glm.gif" alt="GLM"></center>

<hr>

<p>The output of the GLM analysis is produced under the <code class="language-plaintext highlighter-rouge">Result</code> node. The GLM association test can be evaluated by plotting <code class="language-plaintext highlighter-rouge">Q-Q plot</code> and the <code class="language-plaintext highlighter-rouge">Manhattan</code> plot as shown below.</p>

<center>
<img src="/images/biostat/gwas/manhattan.png" width="auto" height="200" alt="GLM Manhattan plot">
<img src="/images/biostat/gwas/qqplot.png" width="auto" height="200" alt="Q-Q plot">
</center>

<p>From the above Q-Q plot, we can see that are several markers that appear to be falsely associated with the trait, therefore, to control this confounding effect, use <code class="language-plaintext highlighter-rouge">Kinship</code> matrix as an another covariate in the linear model</p>

<hr>

<h2 id="40-mixed-linear-model-mlm">4.0 Mixed Linear Model (MLM)</h2>

<p>Mixed Linear Model used <a href="https://www.nature.com/articles/ng1702">Unified Mixed-Model Method for Association Mapping</a>. It helps to reduces Type I error in association mapping with complex pedigrees, families, founding effects and population structure.</p>

<h3 id="41-calculating-kinship-matrix">4.1 Calculating Kinship matrix</h3>
<p>Follow the below steps to calculate the kinship matrix:</p>

<center><img src="/images/biostat/gwas/kinship.gif" alt="Kinship matrix"></center>

<hr>

<h3 id="42-running-mixed-linear-model-mlm">4.2 running Mixed Linear Model (MLM)</h3>
<p>MLM model includes the PCA and the kinship matrix i.e. MLM(PCA+K).</p>

<p>Therefore, once the Kinship matrix has been calculated, MLM can be now be conducted by following below steps:</p>

<center><img src="/images/biostat/gwas/mlm.gif" alt="MLM steps"></center>

<hr>

<p>Plot the output (MLM stats file in the Results branch following the above shown steps).</p>
<center>
<img src="/images/biostat/gwas/manhattanMlm.png" width="auto" height="200" alt="MLM Manhattan plot">
<img src="/images/biostat/gwas/qqmlm.png" width="auto" height="200" alt="MLM Q-Q plot">                                                           
</center>

<hr>

<h3 id="42-exporting-results">4.2 Exporting results</h3>
<p>One may export the results in .txt format by the following the below steps:</p>

<center>
<img src="/images/biostat/gwas/export.gif" alt="Export results">
</center>

<hr>

<h3 id="43-significance-threshold">4.3 Significance Threshold</h3>

<p>Bonferroni threshold can be determined to identify significantly markers associated with the trait  by using the below equation:</p>

<div class="language-scss highlighter-rouge"><div class="highlight"><pre class="highlight"><code><span class="nt">P</span> <span class="err">≤</span> <span class="nt">1</span><span class="o">/</span><span class="nt">N</span> <span class="o">(</span><span class="err">α</span> <span class="o">=</span><span class="nt">0</span><span class="nc">.05</span><span class="o">)</span>
</code></pre></div></div>
<p>where, N is the total number of markers tested in association analysis) was used to identify the most significantly markers associated with the trait. Similarly, another way is to perform <code class="language-plaintext highlighter-rouge">FDR</code> (<code class="language-plaintext highlighter-rouge">False Discovery Rate</code>) correction method.</p>

<hr>
<center> <b>End of Tutorial</b>
</center>

<hr>

<p><strong>Thank you</strong> for reading this tutorial. If you have any query or comments, please let me know in the comment section below or send me an email.</p>

<hr>
<p><b> Bibliography </b></p>
<ol>
<li>
<p>	Bradbury PJ, Zhang Z, Kroon DE, Casstevens TM, Ramdoss Y, Buckler ES. (2007)<cite> TASSEL: Software for association mapping of complex traits in diverse samples.</cite> Bioinformatics 23:2633-2635.</p>
</li>
<li>
<p>Yu, J., Pressoir, G., Briggs, W. et al. (2006) <cite>A unified mixed-model method for association mapping that accounts for multiple levels of relatedness</cite> Nat Genet 38, 203–208. https://doi.org/10.1038/ng1702 </p>
</li>
</ol>

    

      
  
        
      </section>

      <footer class="page__meta">
        
        


  





  

  <p class="page__taxonomy">
    <strong><i class="fas fa-fw fa-tags" aria-hidden="true"></i> Tags: </strong>
    <span itemprop="keywords">
    
      <a href="/tags/#genome-wide-association-analysis" class="page__taxonomy-item p-category" rel="tag">Genome-wide association analysis</a><span class="sep">, </span>
    
      <a href="/tags/#glm" class="page__taxonomy-item p-category" rel="tag">GLM</a><span class="sep">, </span>
    
      <a href="/tags/#gwas-gui" class="page__taxonomy-item p-category" rel="tag">GWAS GUI</a><span class="sep">, </span>
    
      <a href="/tags/#linear-model" class="page__taxonomy-item p-category" rel="tag">Linear model</a><span class="sep">, </span>
    
      <a href="/tags/#mlm" class="page__taxonomy-item p-category" rel="tag">MLM</a><span class="sep">, </span>
    
      <a href="/tags/#tassel" class="page__taxonomy-item p-category" rel="tag">TASSEL</a>
    
    </span>
  </p>




  




  

  <p class="page__taxonomy">
    <strong><i class="fas fa-fw fa-folder-open" aria-hidden="true"></i> Categories: </strong>
    <span itemprop="keywords">
    
      <a href="/categories/#software" class="page__taxonomy-item p-category" rel="tag">software</a><span class="sep">, </span>
    
      <a href="/categories/#tassel" class="page__taxonomy-item p-category" rel="tag">TASSEL</a>
    
    </span>
  </p>


        

  <p class="page__date"><strong><i class="fas fa-fw fa-calendar-alt" aria-hidden="true"></i> Updated:</strong> <time datetime="2024-10-06">October 6, 2024</time></p>


      </footer>

      <section class="page__share">
  <h4 class="page__share-title">Share on</h4>

  <a href="https://twitter.com/intent/tweet?via=DrShamshadM&amp;text=A+tutorial+on+Genome-Wide+Association+Studies+%28GWAS%29+in+Tassel+%28GUI%29%20http%3A%2F%2Flocalhost%3A4000%2Fposts%2F2021%2F01%2FA-tutorial-on-Genome-Wide-Association-Studies%2F" class="btn btn--twitter" onclick="window.open(this.href, 'window', 'left=20,top=20,width=500,height=500,toolbar=1,resizable=0'); return false;" title="Share on Twitter"><i class="fab fa-fw fa-twitter" aria-hidden="true"></i><span> Twitter</span></a>

  <a href="https://www.facebook.com/sharer/sharer.php?u=http%3A%2F%2Flocalhost%3A4000%2Fposts%2F2021%2F01%2FA-tutorial-on-Genome-Wide-Association-Studies%2F" class="btn btn--facebook" onclick="window.open(this.href, 'window', 'left=20,top=20,width=500,height=500,toolbar=1,resizable=0'); return false;" title="Share on Facebook"><i class="fab fa-fw fa-facebook" aria-hidden="true"></i><span> Facebook</span></a>

  <a href="https://www.linkedin.com/shareArticle?mini=true&amp;url=http://localhost:4000/posts/2021/01/A-tutorial-on-Genome-Wide-Association-Studies/" class="btn btn--linkedin" onclick="window.open(this.href, 'window', 'left=20,top=20,width=500,height=500,toolbar=1,resizable=0'); return false;" title="Share on LinkedIn"><i class="fab fa-fw fa-linkedin" aria-hidden="true"></i><span> LinkedIn</span></a>
</section>


      


  <nav class="pagination">
    
      <a href="http://localhost:4000/posts/2021/01/investigate-genetic-admixture-using-STRUCTURE-software/" class="pagination--pager" title="A tutorial on investigate genetic admixture using STRUCTURE software
">Previous</a>
    
    
      <a href="#" class="pagination--pager disabled">Next</a>
    
  </nav>

    </div>

    
  </article>

  
  
    
<div class="page__related">
  <h2 class="page__related-title">You may also enjoy</h2>
  <div class="grid__wrapper">
    
      
      





<div class="grid__item">
  <article class="archive__item" itemscope itemtype="https://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2021/01/investigate-genetic-admixture-using-STRUCTURE-software/" rel="permalink">A tutorial on investigate genetic admixture using STRUCTURE software
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="far fa-clock-o" aria-hidden="true"></i> 


  
	  3 minute read
	
</p>
    
    
    
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar-alt" aria-hidden="true"></i> Updated:</strong> <time datetime="2021-01-05T00:00:00+05:30">January 05, 2021</time></p>
        

  
    <p class="archive__item-excerpt" itemprop="description"></p>
<p><strong>Structure Software</strong> is a freely available software package that one may use for rigorous investigation of <code class="language-plaintext highlighter-rouge">admixed individuals</code>; identification of <code class="language-plaintext highlighter-rouge">point of hybridization</code> and <code class="language-plaintext highlighter-rouge">migrants</code>; and estimate over all <code class="language-plaintext highlighter-rouge">structure</code> of a population using a commonly used genetic markers such as <code class="language-plaintext highlighter-rouge">SNPs</code> and <code class="language-plaintext highlighter-rouge">SSRs</code>. This software was developed by <strong>Pritchard Lab</strong> at <strong>Stanford University</strong> and can downloaded at this <a href="https://web.stanford.edu/group/pritchardlab/structure.html">link</a>.</p>


    

    
    

      <!-- citation and icon code -->  
      
      
      
      
      
	 </article>
</div>

    
      
      





<div class="grid__item">
  <article class="archive__item" itemscope itemtype="https://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2020/10/mapchart/" rel="permalink">Plot Genetic Linkage Maps using MapChart software
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="far fa-clock-o" aria-hidden="true"></i> 


  
	  5 minute read
	
</p>
    
    
    
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar-alt" aria-hidden="true"></i> Updated:</strong> <time datetime="2020-10-10T00:00:00+05:30">October 10, 2020</time></p>
        

  
    <p class="archive__item-excerpt" itemprop="description"></p>
<p><strong>MapChart</strong> is a free software to plot publishing quality genetic linkage maps as well as QTLs. This software was developed at <strong>Wageningen University</strong> by <strong>Roeland E. Voorrips</strong>  and can be downloaded at this <a href="https://www.wur.nl/en/show/Mapchart.htm"> link </a>.</p>


    

    
    

      <!-- citation and icon code -->  
      
      
      
      
      
	 </article>
</div>

    
      
      





<div class="grid__item">
  <article class="archive__item" itemscope itemtype="https://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2020/09/scientific-paper/" rel="permalink">Online submission of a scientific paper, a guideline for beginners
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="far fa-clock-o" aria-hidden="true"></i> 


  
	  5 minute read
	
</p>
    
    
    
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar-alt" aria-hidden="true"></i> Updated:</strong> <time datetime="2020-09-04T00:00:00+05:30">September 04, 2020</time></p>
        

  
    <p class="archive__item-excerpt" itemprop="description"></p>
<p>There exist some definite gaps “from performing of an experiment to the formulation of a hypothesis”. It is not necessary that a person good at lab work must be an expert in scientific writing too.</p>


    

    
    

      <!-- citation and icon code -->  
      
      
      
      
      
	 </article>
</div>

    
      
      





<div class="grid__item">
  <article class="archive__item" itemscope itemtype="https://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2020/08/non-whitelisted-plugin-Mermaid/" rel="permalink">Add non-whitelisted plugins and Mermaid
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="far fa-clock-o" aria-hidden="true"></i> 


  
	  6 minute read
	
</p>
    
    
    
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar-alt" aria-hidden="true"></i> Updated:</strong> <time datetime="2020-08-20T00:00:00+05:30">August 20, 2020</time></p>
        

  
    <p class="archive__item-excerpt" itemprop="description"></p>
<p>GitHub Pages runs in <code class="language-plaintext highlighter-rouge">safe</code> mode and only allows <a href="https://pages.github.com/versions/">a set of whitelisted plugins</a>. To use the gem in GitHub Pages, one of the workarounds is to use CI (e.g. travis, github workflow) and deploy to your <code class="language-plaintext highlighter-rouge">gh-pages</code> branch like: <a href="https://github.com/jeffreytse/jekyll-deploy-action">jekyll-deploy-action</a>, and I use this plugin: <a href="https://github.com/jeffreytse/jekyll-spaceship">jekyll-spaceship</a> in my github pages.</p>


    

    
    

      <!-- citation and icon code -->  
      
      
      
      
      
	 </article>
</div>

    
  </div>
</div>
		  
  
  
</div>

    </div>

    

    <div id="footer" class="page__footer">
      <footer>
        <!-- start custom footer snippets -->
<!-- end custom head snippets -->

<!-- end custom footer snippets -->
        <div class="page__footer-follow">
  <ul class="social-icons">
    
      <li><strong>Follow:</strong></li>
    

    
      
        
          <li><a href="https://twitter.com/DrShamshadM" rel="nofollow noopener noreferrer"><i class="fab fa-fw fa-twitter-square" aria-hidden="true"></i> Twitter</a></li>
        
      
        
          <li><a href="https://scholar.google.com/citations?user=dI7V1a4AAAAJ&amp;hl=en" rel="nofollow noopener noreferrer"><i class="ai ai-google-scholar-square ai-fw" aria-hidden="true"></i> Google Scholar</a></li>
        
      
        
          <li><a href="https://www.researchgate.net/profile/Mohd-Shamshad" rel="nofollow noopener noreferrer"><i class="ai ai-researchgate-square ai-fw" aria-hidden="true"></i> ResearchGate</a></li>
        
      
    

    
      <li><a href="/feed.xml"><i class="fas fa-fw fa-rss-square" aria-hidden="true"></i> Feed</a></li>
    
  </ul>
</div>

<div class="page__footer-copyright">© 2025 M. Shamshad. Proudly hosted by <a href="https://jekyllrb.com" rel="nofollow">Jekyll</a> &amp; <a href="https://mademistakes.com/work/minimal-mistakes-jekyll-theme/" rel="nofollow">Minimal Mistakes</a>.</div>



<link defer rel="preload" href="/assets/js/mermaid.min.js" as="script">
<script defer src="/assets/js/mermaid.min.js"></script>
      </footer>
    </div>

    
  <link defer rel="preload" href="/assets/js/main.min.js" as="script">
  <script defer src="/assets/js/main.min.js"></script>
  <!-- <script src="https://kit.fontawesome.com/4eee35f757.js"></script> -->








  
  <script type="text/javascript">
  	/* * * CONFIGURATION VARIABLES: EDIT BEFORE PASTING INTO YOUR WEBPAGE * * */
  	var disqus_shortname = 'shamshad';

  	/* * * DON'T EDIT BELOW THIS LINE * * */
  	(function() {
  		var dsq = document.createElement('script'); dsq.type = 'text/javascript'; dsq.async = true;
  		dsq.src = '//' + disqus_shortname + '.disqus.com/embed.js';
  		(document.getElementsByTagName('head')[0] || document.getElementsByTagName('body')[0]).appendChild(dsq);
  	})();

  	/* * * DON'T EDIT BELOW THIS LINE * * */
  	(function () {
  		var s = document.createElement('script'); s.async = true;
  		s.type = 'text/javascript';
  		s.src = '//' + disqus_shortname + '.disqus.com/count.js';
  		(document.getElementsByTagName('HEAD')[0] || document.getElementsByTagName('BODY')[0]).appendChild(s);
  	}());
  </script>
  <noscript>Please enable JavaScript to view the <a href="http://disqus.com/?ref_noscript">comments powered by Disqus.</a>
</noscript>






  
  <script defer src="/assets/js/mermaid.js"></script>
  


  </body>
</html>

