<!DOCTYPE html>
<html lang="en" class="no-js">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8"> <!-- begin _includes/seo.html --><title>A tutorial on investigate genetic admixture using STRUCTURE software | M. Shamshad</title>
<meta name="description" content="Structure Software is a freely available software package that one may use for rigorous investigation of admixed individuals; identification of point of hybridization and migrants; and estimate over all structure of a population using a commonly used genetic markers such as SNPs and SSRs. This software was developed by Pritchard Lab at Stanford University and can downloaded at this link.">
<meta name="author" content="M. Shamshad">
<meta property="article:author" content="M. Shamshad">
<meta property="og:type" content="article">
<meta property="og:locale" content="en_US">
<meta property="og:site_name" content="M. Shamshad">
<meta property="og:title" content="A tutorial on investigate genetic admixture using STRUCTURE software">
<meta property="og:url" content="https://shamshad.in/posts/2021/01/investigate-genetic-admixture-using-STRUCTURE-software/">
<meta property="og:description" content="Structure Software is a freely available software package that one may use for rigorous investigation of admixed individuals; identification of point of hybridization and migrants; and estimate over all structure of a population using a commonly used genetic markers such as SNPs and SSRs. This software was developed by Pritchard Lab at Stanford University and can downloaded at this link.">
<meta property="og:image" content="https://shamshad.in/images/biostat/structure/structure-admixture.webp">
<meta name="twitter:site" content="@DrShamshadM">
<meta name="twitter:title" content="A tutorial on investigate genetic admixture using STRUCTURE software">
<meta name="twitter:description" content="Structure Software is a freely available software package that one may use for rigorous investigation of admixed individuals; identification of point of hybridization and migrants; and estimate over all structure of a population using a commonly used genetic markers such as SNPs and SSRs. This software was developed by Pritchard Lab at Stanford University and can downloaded at this link.">
<meta name="twitter:url" content="https://shamshad.in/posts/2021/01/investigate-genetic-admixture-using-STRUCTURE-software/">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:image" content="https://shamshad.in/images/biostat/structure/structure-admixture.webp">
<meta name="twitter:creator" content="@DrShamshadM">
<meta property="article:published_time" content="2021-01-05T00:00:00+05:30">
<meta property="article:modified_time" content="2023-04-10T17:04:21+05:30">
<link rel="canonical" href="https://shamshad.in/posts/2021/01/investigate-genetic-admixture-using-STRUCTURE-software/">
<meta name="google-site-verification" content="BD1FGtpPEYJphVV41YzUn-mrOqREjhO-JsWa4YZgj60"> <!-- end _includes/seo.html --><link href="/feed.xml" type="application/atom+xml" rel="alternate" title="M. Shamshad Feed"> <!-- https://web.dev/viewport/?utm_source=lighthouse&utm_medium=lr --><meta name="viewport" content="width=device-width, initial-scale=1.0"> <script async> document.documentElement.className = document.documentElement.className.replace(/\bno-js\b/g, "") + " js "; </script><link rel="preload" as="style" onload="this.rel = 'stylesheet'" href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&amp;display=swap"> <noscript>
<link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&amp;display=swap"> </noscript> <!-- For all browsers --><link rel="stylesheet" href="/assets/css/main.css" id="theme_source">
<link rel="stylesheet alternate" href="/assets/css/mainDark.css" id="theme_source_2"> <script async> let theme = sessionStorage.getItem("theme"); if (theme === "dark") { sessionStorage.setItem("theme", "dark"); node1 = document.getElementById("theme_source"); node2 = document.getElementById("theme_source_2"); node1.setAttribute("rel", "stylesheet alternate"); node2.setAttribute("rel", "stylesheet"); } else { sessionStorage.setItem("theme", "light"); } </script> <!-- start custom head snippets --><meta name="apple-mobile-web-app-capable" content="yes">
<meta name="apple-mobile-web-app-status-bar-style" content="default">
<link rel="apple-touch-icon" sizes="60x60" href="/images/favicon/apple-touch-icon-60x60.png?v=565802">
<link rel="apple-touch-icon" sizes="76x76" href="/images/favicon/apple-touch-icon-76x76.png?v=565802">
<link rel="apple-touch-icon" sizes="120x120" href="/images/favicon/apple-touch-icon-120x120.png?v=565802">
<link rel="apple-touch-icon" sizes="152x152" href="/images/favicon/apple-touch-icon-152x152.png?v=565802">
<link rel="apple-touch-icon" sizes="180x180" href="/images/favicon/apple-touch-icon-180x180.png?v=565802">
<link rel="icon" type="image/png" sizes="32x32" href="/images/favicon/favicon-32x32.png?v=565802">
<link rel="icon" type="image/png" sizes="16x16" href="/images/favicon/favicon-16x16.png?v=565802">
<link rel="manifest" href="/images/favicon/site.webmanifest?v=565802">
<link rel="mask-icon" href="/images/favicon/safari-pinned-tab.svg?v=565802" color="#00a7ff">
<link rel="shortcut icon" href="/images/favicon/favicon.ico?v=565802">
<meta name="apple-mobile-web-app-title" content="SM-Web">
<meta name="application-name" content="SM-Web">
<meta name="msapplication-TileColor" content="#00a300">
<meta name="msapplication-config" content="/images/favicon/browserconfig.xml?v=565802">
<meta name="theme-color" content="#ffffff"> <!-- SEO tags --> <!-- Begin Jekyll SEO tag v2.8.0 --><title>A tutorial on investigate genetic admixture using STRUCTURE software | M. Shamshad</title>
<meta name="generator" content="Jekyll v4.3.4">
<meta property="og:title" content="A tutorial on investigate genetic admixture using STRUCTURE software">
<meta name="author" content="M. Shamshad">
<meta property="og:locale" content="en_US">
<meta name="description" content="Structure Software is a freely available software package that one may use for rigorous investigation of admixed individuals; identification of point of hybridization and migrants; and estimate over all structure of a population using a commonly used genetic markers such as SNPs and SSRs. This software was developed by Pritchard Lab at Stanford University and can downloaded at this link.">
<meta property="og:description" content="Structure Software is a freely available software package that one may use for rigorous investigation of admixed individuals; identification of point of hybridization and migrants; and estimate over all structure of a population using a commonly used genetic markers such as SNPs and SSRs. This software was developed by Pritchard Lab at Stanford University and can downloaded at this link.">
<link rel="canonical" href="https://shamshad.in/posts/2021/01/investigate-genetic-admixture-using-STRUCTURE-software/">
<meta property="og:url" content="https://shamshad.in/posts/2021/01/investigate-genetic-admixture-using-STRUCTURE-software/">
<meta property="og:site_name" content="M. Shamshad">
<meta property="og:type" content="article">
<meta property="article:published_time" content="2021-01-05T00:00:00+05:30">
<meta name="twitter:card" content="summary">
<meta property="twitter:title" content="A tutorial on investigate genetic admixture using STRUCTURE software">
<meta name="twitter:site" content="@DrShamshadM">
<meta name="twitter:creator" content="@DrShamshadM">
<meta name="google-site-verification" content="BD1FGtpPEYJphVV41YzUn-mrOqREjhO-JsWa4YZgj60"> <script type="application/ld+json"> {"@context":"https://schema.org","@type":"BlogPosting","author":{"@type":"Person","name":"M. Shamshad"},"dateModified":"2023-04-10T17:04:21+05:30","datePublished":"2021-01-05T00:00:00+05:30","description":"Structure Software is a freely available software package that one may use for rigorous investigation of admixed individuals; identification of point of hybridization and migrants; and estimate over all structure of a population using a commonly used genetic markers such as SNPs and SSRs. This software was developed by Pritchard Lab at Stanford University and can downloaded at this link.","headline":"A tutorial on investigate genetic admixture using STRUCTURE software","mainEntityOfPage":{"@type":"WebPage","@id":"https://shamshad.in/posts/2021/01/investigate-genetic-admixture-using-STRUCTURE-software/"},"publisher":{"@type":"Organization","logo":{"@type":"ImageObject","url":"https://shamshad.in/images/favicon/site-treaser.webp"},"name":"M. Shamshad"},"url":"https://shamshad.in/posts/2021/01/investigate-genetic-admixture-using-STRUCTURE-software/"}</script> <!-- End Jekyll SEO tag -->
</head>
<body class="layout--single">
<nav class="skip-links"><ul>
<li><a href="#site-nav" class="screen-reader-shortcut">Skip to primary navigation</a></li>
<li><a href="#main" class="screen-reader-shortcut">Skip to content</a></li>
<li><a href="#footer" class="screen-reader-shortcut">Skip to footer</a></li>
</ul></nav><div class="masthead"><div class="masthead__inner-wrap"><div class="masthead__menu"><nav id="site-nav" class="greedy-nav"> <a class="site-logo" href="/"><img src="/images/favicon/site-treaser.webp" alt="M. Shamshad"></a> <a class="site-title" href="/"> M. Shamshad </a><ul class="visible-links">
<li class="masthead__menu-item"> <a href="/about/">About</a>
</li>
<li class="masthead__menu-item"> <a href="/teaching/">Teaching</a>
</li>
<li class="masthead__menu-item"> <a href="/publications/">Publications</a>
</li>
<li class="masthead__menu-item"> <a href="/talks/">Talks</a>
</li>
<li class="masthead__menu-item"> <a href="/research/">Research</a>
</li>
<li class="masthead__menu-item"> <a href="/resume/">Resume</a>
</li>
<li class="masthead__menu-item"> <a href="/tags/">Tags</a>
</li>
<li class="masthead__menu-item"> <a href="/docs/Linux_tutorial/">Bioinformatics</a>
</li>
</ul>
<span class="fa-solid fa-moon toggle-switch" type="button" style="cursor: pointer;" aria-hidden="true" onclick="node1 = document.getElementById('theme_source'); node2 = document.getElementById('theme_source_2'); if(node1.getAttribute('rel')=='stylesheet'){node1.setAttribute('rel', 'stylesheet alternate'); node2.setAttribute('rel', 'stylesheet'); sessionStorage.setItem('theme', 'dark');} else{node2.setAttribute('rel', 'stylesheet alternate'); node1.setAttribute('rel', 'stylesheet');sessionStorage.setItem('theme', 'light');} return false;"></span> <button class="greedy-nav__toggle hidden" type="button"> <span class="visually-hidden">Toggle menu</span><div class="navicon"></div></button><ul class="hidden-links hidden"></ul></nav></div></div></div>
<div class="initial-content"><div id="main" role="main">
<div class="sidebar sticky"><div itemscope itemtype="https://schema.org/Person" class="h-card">
<div class="author__avatar"> <a href="https://shamshad.in/"> <img fetchpriority="high" src="/images/profile.jpeg" alt="M. Shamshad" itemprop="image" class="u-photo"> </a>
</div>
<div class="author__content">
<h3 class="author__name p-name" itemprop="name"> <a class="u-url" rel="me" href="https://shamshad.in/" itemprop="url">M. Shamshad</a>
</h3>
<div class="author__bio p-note" itemprop="description"><p>Plant Breeding &amp;&amp; Data Science</p></div>
</div>
<div class="author__urls-wrapper"> <button class="btn btn--inverse">Follow</button><ul class="author__urls social-icons">
<li itemprop="homeLocation" itemscope itemtype="https://schema.org/Place"> <i class="fas fa-fw fa-map-marker-alt" aria-hidden="true"></i> <span itemprop="name" class="p-locality">Ludhina, Punjab, India <img class="emoji" title=":india:" alt=":india:" src="https://github.githubassets.com/images/icons/emoji/unicode/1f1ee-1f1f3.png" height="20" width="20"></span>
</li>
<li> <a href="mailto:shamshad.rattan@gmail.com" rel="me" class="u-email"><meta itemprop="email" content="shamshad.rattan@gmail.com"> <i class="fas fa-fw fa-envelope" aria-hidden="true"></i><span class="label">Email</span> </a>
</li>
<li><a href="https://www.researchgate.net/profile/Mohd-Shamshad" itemprop="sameAs" rel="nofollow noopener noreferrer me"> <i class="ai ai-researchgate-square" aria-hidden="true"></i><span class="label">ResearchGate</span> </a></li>
<li> <a href="https://twitter.com/DrShamshadM" itemprop="sameAs" rel="nofollow noopener noreferrer me"> <i class="fab fa-fw fa-twitter-square" aria-hidden="true"></i><span class="label">Twitter</span> </a>
</li>
<li> <a href="https://www.linkedin.com/in/dr-m-shamshad-58b4321b1" itemprop="sameAs" rel="nofollow noopener noreferrer me"> <i class="fab fa-fw fa-linkedin" aria-hidden="true"></i><span class="label">Linkedin</span> </a>
</li>
<li><a href="https://scholar.google.com/citations?user=dI7V1a4AAAAJ&hl=en" itemprop="sameAs" rel="nofollow noopener noreferrer me"> <i class="ai ai-fw ai-google-scholar-square" aria-hidden="true"></i><span class="label">Google Scholar</span></a></li>
<!--<li> <a href="https://link-to-whatever-social-network.com/user/" itemprop="sameAs" rel="nofollow noopener noreferrer"> <i class="fas fa-fw" aria-hidden="true"></i> Custom Social Profile Link </a></li>-->
</ul>
</div>
</div></div>
<article class="page" itemscope itemtype="https://schema.org/CreativeWork"><meta itemprop="headline" content="A tutorial on investigate genetic admixture using STRUCTURE software">
<meta itemprop="description" content="Structure Software is a freely available software package that one may use for rigorous investigation of admixed individuals; identification of point of hybridization and migrants; and estimate over all structure of a population using a commonly used genetic markers such as SNPs and SSRs. This software was developed by Pritchard Lab at Stanford University and can downloaded at this link.">
<meta itemprop="datePublished" content="2021-01-05T00:00:00+05:30">
<meta itemprop="dateModified" content="2023-04-10T17:04:21+05:30">
<div class="page__inner-wrap">
<header><h1 id="page-title" class="page__title" itemprop="headline"> <a href="https://shamshad.in/posts/2021/01/investigate-genetic-admixture-using-STRUCTURE-software/" itemprop="url">A tutorial on investigate genetic admixture using STRUCTURE software </a>
</h1>
<p class="page__meta"> <span class="page__meta-readtime"> <i class="far fa-clock" aria-hidden="true"></i> 3 minute read </span></p></header><section class="page__content" itemprop="text"><aside class="sidebar__right "><nav class="toc"><header><h4 class="nav__title">
<i class="fas fa-file-alt"></i> On this page</h4></header><ul class="toc__menu">
<li><a href="#step-1-preparing-the-input-file">Step 1: Preparing the Input file</a></li>
<li>
<a href="#step-2-running-the-structure-biostat">Step 2: Running the Structure biostat</a><ul>
<li><a href="#11-importing-input-file">1.1 Importing input file</a></li>
<li><a href="#step-12-set-parameter">Step: 1.2 Set Parameter</a></li>
<li><a href="#step-13-running-the-project">Step: 1.3 Running the project</a></li>
<li><a href="#step-14-viewing-the-results">Step: 1.4 Viewing the results</a></li>
</ul>
</li>
<li><a href="#21-preparing-files-for-structure-harvester">2.1 Preparing Files for Structure Harvester</a></li>
<li><a href="#22-running-structure-harvester">2.2 Running Structure Harvester</a></li>
<li><a href="#23-interpreting-the-output">2.3 Interpreting the output</a></li>
</ul></nav></aside><p><strong>Structure Software</strong> is a freely available software package that one may use for rigorous investigation of <code class="language-plaintext highlighter-rouge">admixed individuals</code>; identification of <code class="language-plaintext highlighter-rouge">point of hybridization</code> and <code class="language-plaintext highlighter-rouge">migrants</code>; and estimate over all <code class="language-plaintext highlighter-rouge">structure</code> of a population using a commonly used genetic markers such as <code class="language-plaintext highlighter-rouge">SNPs</code> and <code class="language-plaintext highlighter-rouge">SSRs</code>. This software was developed by <strong>Pritchard Lab</strong> at <strong>Stanford University</strong> and can downloaded at this <a href="https://web.stanford.edu/group/pritchardlab/structure.html">link</a>. <!--more--></p>
<hr>
<p>Download sample data set: <a href="/images/biostat/structure/data_structure_1.txt">click here</a></p>
<hr>
<p>In this tutorial, I will show how to prepare <code class="language-plaintext highlighter-rouge">input</code> files and run the <code class="language-plaintext highlighter-rouge">Structure</code> software. For detail information, please read this article at this <a href="https://web.stanford.edu/group/pritchardlab/structure.html">link</a></p>
<hr>
<h2 id="step-1-preparing-the-input-file">Step 1: Preparing the Input file</h2>
<p>In this tutorial, I am using <code class="language-plaintext highlighter-rouge">numerical</code> SNP data as in <code class="language-plaintext highlighter-rouge">input</code> genotype file. One can convert their genotype data in numerical format in <code class="language-plaintext highlighter-rouge">TASSEL</code> software or any software package available as per ones convenience. The file needs to be formatted properly as shown below in the image below and save it as <code class="language-plaintext highlighter-rouge">.txt</code> file.</p>
<p align="center"> <img src="/images/biostat/structure/input-structure.webp" alt="Input structure File"></p>
<hr>
<p><strong>Please Note</strong> Missing data is denoted as <code class="language-plaintext highlighter-rouge">-9</code> in the above image.</p>
<h2 id="step-2-running-the-structure-biostat">Step 2: Running the Structure biostat</h2>
<!--more--><h3 id="11-importing-input-file">1.1 Importing input file</h3>
<p>Once the input file with the correct header and format is ready, import the the file in <code class="language-plaintext highlighter-rouge">Structure</code> software using the steps shown in the below figure. The importing steps include <strong>4</strong> steps, please make sure to select correct directory and file name. At <strong>step 2 of 4</strong> make sure to correctly input <code class="language-plaintext highlighter-rouge">number of markers</code>, <code class="language-plaintext highlighter-rouge">samples/individual</code>, and <code class="language-plaintext highlighter-rouge">ploidy</code> (if genotypes are ‘A’ then enter 1 but if, it is ‘AA’ enter 2), and finally, enter how the <code class="language-plaintext highlighter-rouge">missing data</code> are indicated as in the file. In this tutorial, I denoting the missing data ‘-9’.</p>
<hr>
<p align="center"> <img src="/images/biostat/structure/Import-structure.gif" alt="Import data in structure"></p>
<hr>
<h3 id="step-12-set-parameter">Step: 1.2 Set Parameter</h3>
<p>Follow the steps shown in the below figure to run this step. <strong>Please remember</strong> One make custom add the <code class="language-plaintext highlighter-rouge">length of burning period</code> and <code class="language-plaintext highlighter-rouge">Number of MCMC Reps after burnin</code>.</p>
<hr>
<p align="center"> <img src="/images/biostat/structure/Set-Parameter.gif" alt="Set parameters structure"></p>
<hr>
<h3 id="step-13-running-the-project">Step: 1.3 Running the project</h3>
<p>Follow the steps shown in the below figure to run this step. <strong>Please remember</strong> to run at least 10 <code class="language-plaintext highlighter-rouge">number of iterations</code>. One see the job progress at the bottom black window of the shell.</p>
<hr>
<p align="center"> <img src="/images/biostat/structure/run-structure-project.gif" alt="Running the project structure"></p>
<hr>
<h3 id="step-14-viewing-the-results">Step: 1.4 Viewing the results</h3>
<p>Follow the steps shown in the below figure to run this step. <strong>Please remember</strong> under the <code class="language-plaintext highlighter-rouge">Results</code> folder there are several branches of the results with various <code class="language-plaintext highlighter-rouge">k</code> values, which indicates <strong>number of sub-populations</strong> estimated from the given genetic data. It can tricky to pick the correct number of <code class="language-plaintext highlighter-rouge">k</code> for the data, and to solve this follow the next step to prepare files for a different software known as <code class="language-plaintext highlighter-rouge">Structure Harvester</code>.</p>
<hr>
<p align="center"> <img src="/images/biostat/structure/structure-resultus.gif" alt="Viewing the structure results"></p>
<!--more--><h2 id="21-preparing-files-for-structure-harvester">2.1 Preparing Files for Structure Harvester</h2>
<p><code class="language-plaintext highlighter-rouge">zip</code> all the result files in the results folder. <!--more--></p>
<p align="center"> <img src="/images/biostat/structure/structure-harvester.gif" alt="Files for Structure Harvester"></p>
<hr>
<h2 id="22-running-structure-harvester">2.2 Running Structure Harvester</h2>
<p>One your web browser search for <code class="language-plaintext highlighter-rouge">structure harvester</code>, and click the first the search result. Next, upload the <code class="language-plaintext highlighter-rouge">results.zip</code> file, click <code class="language-plaintext highlighter-rouge">harvest</code> to run the Structure Harvester program. It can take about few mins to run, however, it definitely depends on your data. Once the job is completed, the program outputs the summary of the analysis, the key output to look at is <code class="language-plaintext highlighter-rouge">Delta K</code> plot and <code class="language-plaintext highlighter-rouge">Evanno table</code>.</p>
<hr>
<p align="center"> <img src="/images/biostat/structure/run-structure-harvester.gif" alt="Run Structure Harvester"></p>
<!--more--><h2 id="23-interpreting-the-output">2.3 Interpreting the output</h2>
<p><code class="language-plaintext highlighter-rouge">Evanno table</code> highlights the significant <code class="language-plaintext highlighter-rouge">k</code> value that is estimated for this genotype data (see below figure). For this tutorial data set, the estimated from <code class="language-plaintext highlighter-rouge">k</code> is 3 subpopulation which is also supported by the <code class="language-plaintext highlighter-rouge">Delta K</code> plot, where a clear peak is see at <code class="language-plaintext highlighter-rouge">K = 3</code> (see Delta K plot below).</p>
<p align="center"> <img src="/images/biostat/structure/ktable.webp"></p>
<!--more--><p align="center"> <img src="/images/biostat/structure/deltaK.webp"></p>
<p><strong>Therefore</strong> the correct bar plot with correct number of sub-population (k) is 3, which can be plotted by following the steps shown in <strong>1.4</strong></p>
<p align="center"> <img src="/images/biostat/structure/structure-result-final.webp"></p>
<hr>
<p><strong>Thank you</strong> for reading this tutorial. If you have any questions or comments, please let me know by email.</p>
<hr>
<p>Happy Structure-ing <img class="emoji" title=":smiley:" alt=":smiley:" src="https://github.githubassets.com/images/icons/emoji/unicode/1f603.png" height="20" width="20"></p>
<hr>
<p><b> Bibliography </b></p>
<ol>
<li><p>Pritchard, Jonathan K., William Wen, and Daniel Falush. <cite>"Documentation for STRUCTURE software: Version 2."</cite> (2003).</p></li>
<li><p>Earl, Dent A. <cite> "STRUCTURE HARVESTER: a website and program for visualizing STRUCTURE output and implementing the Evanno method."</cite> Conservation genetics resources 4.2 (2012): 359-361.</p></li>
</ol></section><footer class="page__meta"><p class="page__taxonomy"> <strong><i class="fas fa-fw fa-tags" aria-hidden="true"></i> Tags: </strong> <span itemprop="keywords"> <a href="/tags/#genetic-admixture" class="page__taxonomy-item p-category" rel="tag">Genetic Admixture</a><span class="sep">, </span> <a href="/tags/#genetic-data" class="page__taxonomy-item p-category" rel="tag">Genetic data</a><span class="sep">, </span> <a href="/tags/#population-structure" class="page__taxonomy-item p-category" rel="tag">Population structure</a><span class="sep">, </span> <a href="/tags/#structure-software" class="page__taxonomy-item p-category" rel="tag">Structure software</a> </span></p>
<p class="page__taxonomy"> <strong><i class="fas fa-fw fa-folder-open" aria-hidden="true"></i> Categories: </strong> <span itemprop="keywords"> <a href="/categories/#software" class="page__taxonomy-item p-category" rel="tag">software</a><span class="sep">, </span> <a href="/categories/#structure" class="page__taxonomy-item p-category" rel="tag">Structure</a> </span></p>
<p class="page__date"><strong><i class="fas fa-fw fa-calendar-alt" aria-hidden="true"></i> Updated:</strong> <time datetime="2023-04-10">April 10, 2023</time></p></footer><section class="page__share"><h4 class="page__share-title">Share on</h4>
<a href="https://twitter.com/intent/tweet?via=DrShamshadM&text=A+tutorial+on+investigate+genetic+admixture+using+STRUCTURE+software%20https%3A%2F%2Fshamshad.in%2Fposts%2F2021%2F01%2Finvestigate-genetic-admixture-using-STRUCTURE-software%2F" class="btn btn--twitter" onclick="window.open(this.href, 'window', 'left=20,top=20,width=500,height=500,toolbar=1,resizable=0'); return false;" title="Share on Twitter"><i class="fab fa-fw fa-twitter" aria-hidden="true"></i><span> Twitter</span></a> <a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fshamshad.in%2Fposts%2F2021%2F01%2Finvestigate-genetic-admixture-using-STRUCTURE-software%2F" class="btn btn--facebook" onclick="window.open(this.href, 'window', 'left=20,top=20,width=500,height=500,toolbar=1,resizable=0'); return false;" title="Share on Facebook"><i class="fab fa-fw fa-facebook" aria-hidden="true"></i><span> Facebook</span></a> <a href="https://www.linkedin.com/shareArticle?mini=true&url=https://shamshad.in/posts/2021/01/investigate-genetic-admixture-using-STRUCTURE-software/" class="btn btn--linkedin" onclick="window.open(this.href, 'window', 'left=20,top=20,width=500,height=500,toolbar=1,resizable=0'); return false;" title="Share on LinkedIn"><i class="fab fa-fw fa-linkedin" aria-hidden="true"></i><span> LinkedIn</span></a></section><nav class="pagination"> <a href="https://shamshad.in/posts/2020/10/mapchart/" class="pagination--pager" title="Plot Genetic Linkage Maps using MapChart software ">Previous</a> <a href="https://shamshad.in/posts/2021/01/A-tutorial-on-Genome-Wide-Association-Studies/" class="pagination--pager" title="A tutorial on Genome-Wide Association Studies (GWAS) in Tassel (GUI) ">Next</a></nav>
</div>
<div class="page__comments">
<h4 class="page__comments-title">Leave a comment</h4>
<section id="disqus_thread"></section>
</div></article><div class="page__related">
<h2 class="page__related-title">You may also enjoy</h2>
<div class="grid__wrapper">
<div class="grid__item"><article class="archive__item" itemscope itemtype="https://schema.org/CreativeWork"><h2 class="archive__item-title" itemprop="headline"> <a href="https://shamshad.in/posts/2021/01/A-tutorial-on-Genome-Wide-Association-Studies/" rel="permalink">A tutorial on Genome-Wide Association Studies (GWAS) in Tassel (GUI) </a>
</h2>
<p class="page__meta"><i class="far fa-clock-o" aria-hidden="true"></i> 3 minute read</p>
<p class="page__date"><strong><i class="fa fa-fw fa-calendar-alt" aria-hidden="true"></i> Updated:</strong> <time datetime="2021-01-20T00:00:00+05:30">January 20, 2021</time></p>
<p class="archive__item-excerpt" itemprop="description"></p>
<p>Genome-wide association studies (GWAS) increase their popularity among medical, biological, and social sciences to identify the association between single nucleotide polymorphisms and phenotypic traits. This tutorial aims to provide a guidelines for conducing genome wide analysis in Tassel.</p>
<!-- citation and icon code --></article></div>
<div class="grid__item"><article class="archive__item" itemscope itemtype="https://schema.org/CreativeWork"><h2 class="archive__item-title" itemprop="headline"> <a href="https://shamshad.in/posts/2020/10/mapchart/" rel="permalink">Plot Genetic Linkage Maps using MapChart software </a>
</h2>
<p class="page__meta"><i class="far fa-clock-o" aria-hidden="true"></i> 5 minute read</p>
<p class="page__date"><strong><i class="fa fa-fw fa-calendar-alt" aria-hidden="true"></i> Updated:</strong> <time datetime="2020-10-10T00:00:00+05:30">October 10, 2020</time></p>
<p class="archive__item-excerpt" itemprop="description"></p>
<p><strong>MapChart</strong> is a free software to plot publishing quality genetic linkage maps as well as QTLs. This software was developed at <strong>Wageningen University</strong> by <strong>Roeland E. Voorrips</strong> and can be downloaded at this <a href="https://www.wur.nl/en/show/Mapchart.htm"> link </a>.</p>
<!-- citation and icon code --></article></div>
<div class="grid__item"><article class="archive__item" itemscope itemtype="https://schema.org/CreativeWork"><h2 class="archive__item-title" itemprop="headline"> <a href="https://shamshad.in/posts/2020/09/scientific-paper/" rel="permalink">Online submission of a scientific paper, a guideline for beginners </a>
</h2>
<p class="page__meta"><i class="far fa-clock-o" aria-hidden="true"></i> 5 minute read</p>
<p class="page__date"><strong><i class="fa fa-fw fa-calendar-alt" aria-hidden="true"></i> Updated:</strong> <time datetime="2020-09-04T00:00:00+05:30">September 04, 2020</time></p>
<p class="archive__item-excerpt" itemprop="description"></p>
<p>There exist some definite gaps “from performing of an experiment to the formulation of a hypothesis”. It is not necessary that a person good at lab work must be an expert in scientific writing too.</p>
<!-- citation and icon code --></article></div>
<div class="grid__item"><article class="archive__item" itemscope itemtype="https://schema.org/CreativeWork"><h2 class="archive__item-title" itemprop="headline"> <a href="https://shamshad.in/posts/2020/08/non-whitelisted-plugin-Mermaid/" rel="permalink">Add non-whitelisted plugins and Mermaid </a>
</h2>
<p class="page__meta"><i class="far fa-clock-o" aria-hidden="true"></i> 4 minute read</p>
<p class="page__date"><strong><i class="fa fa-fw fa-calendar-alt" aria-hidden="true"></i> Updated:</strong> <time datetime="2020-08-20T00:00:00+05:30">August 20, 2020</time></p>
<p class="archive__item-excerpt" itemprop="description"></p>
<p>GitHub Pages runs in <code class="language-plaintext highlighter-rouge">safe</code> mode and only allows <a href="https://pages.github.com/versions/">a set of whitelisted plugins</a>. To use the gem in GitHub Pages, one of the workarounds is to use CI (e.g. travis, github workflow) and deploy to your <code class="language-plaintext highlighter-rouge">gh-pages</code> branch like: <a href="https://github.com/jeffreytse/jekyll-deploy-action">jekyll-deploy-action</a>, and I use this plugin: <a href="https://github.com/jeffreytse/jekyll-spaceship">jekyll-spaceship</a> in my github pages.</p>
<!-- citation and icon code --></article></div>
</div>
</div>
</div></div>
<div id="footer" class="page__footer"><footer> <!-- start custom footer snippets --> <script type="text/x-mathjax-config"> MathJax.Hub.Config({ TeX: { equationNumbers: { autoNumber: "all" } } }); </script> <script type="text/x-mathjax-config"> MathJax.Hub.Config({ TeX: { equationNumbers: { autoNumber: "AMS" } }, tex2jax: { inlineMath: [ ['$','$'], ["\\(","\\)"] ], processEscapes: true } }); </script><link rel="preload"> <script async src="https://cdnjs.cloudflare.com/ajax/libs/mathjs/13.1.1/math.js" integrity="sha512-fMI/ndUubOcpzblrAaOFszYIE2iDKua+Cxg9dZp3cAkhsRWEHHbKtFCElHEQ50viX502yj7nFEEcxq2SOTNrIg==" crossorigin="anonymous" referrerpolicy="no-referrer"></script> <!-- end custom head snippets --> <!-- end custom footer snippets --><div class="page__footer-follow"><ul class="social-icons">
<li><strong>Follow:</strong></li>
<li><a href="https://twitter.com/DrShamshadM" rel="nofollow noopener noreferrer"><i class="fab fa-fw fa-twitter-square" aria-hidden="true"></i> Twitter</a></li>
<li><a href="https://scholar.google.com/citations?user=dI7V1a4AAAAJ&hl=en" rel="nofollow noopener noreferrer"><i class="ai ai-google-scholar-square ai-fw" aria-hidden="true"></i> Google Scholar</a></li>
<li><a href="https://www.researchgate.net/profile/Mohd-Shamshad" rel="nofollow noopener noreferrer"><i class="ai ai-researchgate-square ai-fw" aria-hidden="true"></i> ResearchGate</a></li>
<li><a href="/feed.xml"><i class="fas fa-fw fa-rss-square" aria-hidden="true"></i> Feed</a></li>
</ul></div>
<div class="page__footer-copyright">
<img class="emoji" title=":copyright:" alt=":copyright:" src="https://github.githubassets.com/images/icons/emoji/unicode/00a9.png" height="20" width="20"> 2024 M. Shamshad. Proudly hosted by <a href="https://github.com/" rel="nofollow">GitHub</a> &amp; <a href="https://shamshadpbg.github.io/" rel="prev:"><i class="ai ai-academia"></i>mmTheme</a>.</div></footer></div>
<link rel="preload" href="/assets/js/main.min.js" as="script"> <script src="/assets/js/main.min.js" async></script> <!-- <script src="https://kit.fontawesome.com/4eee35f757.js"></script> --> <!-- Global site tag (gtag.js) - Google Analytics --> <script async src="https://www.googletagmanager.com/gtag/js?id=G-H8020H63F5"></script> <script> window.dataLayer = window.dataLayer || []; function gtag(){dataLayer.push(arguments);} gtag('js', new Date()); gtag('config', 'G-H8020H63F5', { 'optimize_id': '', 'anonymize_ip': 'true', }); </script> <script type="text/javascript"> /* * * CONFIGURATION VARIABLES: EDIT BEFORE PASTING INTO YOUR WEBPAGE * * */ var disqus_shortname = 'shamshad'; /* * * DON'T EDIT BELOW THIS LINE * * */ (function() { var dsq = document.createElement('script'); dsq.type = 'text/javascript'; dsq.async = true; dsq.src = '//' + disqus_shortname + '.disqus.com/embed.js'; (document.getElementsByTagName('head')[0] || document.getElementsByTagName('body')[0]).appendChild(dsq); })(); /* * * DON'T EDIT BELOW THIS LINE * * */ (function () { var s = document.createElement('script'); s.async = true; s.type = 'text/javascript'; s.src = '//' + disqus_shortname + '.disqus.com/count.js'; (document.getElementsByTagName('HEAD')[0] || document.getElementsByTagName('BODY')[0]).appendChild(s); }()); </script> <noscript>Please enable JavaScript to view the <a href="http://disqus.com/?ref_noscript">comments powered by Disqus.</a>
</noscript>
</body>
</html>
