<!DOCTYPE html>
<html lang="en" class="no-js">
  <head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
    <meta charset="utf-8">

<!-- begin _includes/seo.html --><title>Morphological and molecular genetic diversity analyses in Helianthus annuus (L.) | M. Shamshad</title>
<meta name="description" content="The knowledge of genetic diversity among parental lines is a prerequisite for selecting parents for hybridization in any heterosis breeding programme. Evaluation of 13 parental lines using D2 analysis was undertaken during spring 2014 at PAU, Ludhiana. The genotypes were grouped into four clusters. Cluster II comprised of maximum number of genotypes (5) viz., CMS-47A, P 93R, P145R, P103R, P 167R followed by cluster I (4 genotypes i.e. CMS-11A, CMS-67A, CMS-234A, 95-C-1); cluster III and IV having two genotypes each i.e. CMS-68A, RCR 8297 and P 124R, P 134R respectively. Maximum inter cluster distance of 578.187 was recorded between genotypes of cluster II and IV while minimum inter cluster distance (175.195) was observed between genotypes present in cluster III and IV. Thirty SSR markers were used for molecular markers analysis. Total number of alleles amplified by 26 polymorphic primers was 51 with an average of 1.96 alleles per locus. The average value of polymorphic information content (PIC) for all the 26 polymorphic markers was 0.47. A polygenetic tree generated, based on Dice dissimilarity matrix and cluster analysis, led to the clustering of the sunflower genotypes into two major groups (one group comprising R lines while other comprising A lines) while one genotype (CMS 234A) formed separate independent cluster. The result of dissimilarity matrix revealed high genetic diversity among all the inbreds lines. Overall, the values for genetic distances ranged from 0.13 to 0.71. The highest genetic distance (0.71) was observed between the genotypes i.e. CMS 67A and P 93R followed by 0.70 for CMS 67A and P124R. Minimum genetic distance 0.13 was observed between P 103R and P 134R, P 134R and P 145R which was indicative of common parentage of these genotypes in the pedigree tree. The principle coordinated analysis along with pattern of clustering of dissimilarity matrix separated the lines into two groups; one having B-lines and other for R-lines. The analysis of molecular variance (AMOVA) showed greater variation within genotypes (68%) as compared to between genotypes (32%).">


  <meta name="author" content="M. Shamshad">
  
  <meta property="article:author" content="M. Shamshad">
  


<meta property="og:type" content="article">
<meta property="og:locale" content="en_US">
<meta property="og:site_name" content="M. Shamshad">
<meta property="og:title" content="Morphological and molecular genetic diversity analyses in Helianthus annuus (L.)">
<meta property="og:url" content="http://localhost:4000/publication/2016-01-12-Morphological-and-Genetic-diversity-in-sunflower">


  <meta property="og:description" content="The knowledge of genetic diversity among parental lines is a prerequisite for selecting parents for hybridization in any heterosis breeding programme. Evaluation of 13 parental lines using D2 analysis was undertaken during spring 2014 at PAU, Ludhiana. The genotypes were grouped into four clusters. Cluster II comprised of maximum number of genotypes (5) viz., CMS-47A, P 93R, P145R, P103R, P 167R followed by cluster I (4 genotypes i.e. CMS-11A, CMS-67A, CMS-234A, 95-C-1); cluster III and IV having two genotypes each i.e. CMS-68A, RCR 8297 and P 124R, P 134R respectively. Maximum inter cluster distance of 578.187 was recorded between genotypes of cluster II and IV while minimum inter cluster distance (175.195) was observed between genotypes present in cluster III and IV. Thirty SSR markers were used for molecular markers analysis. Total number of alleles amplified by 26 polymorphic primers was 51 with an average of 1.96 alleles per locus. The average value of polymorphic information content (PIC) for all the 26 polymorphic markers was 0.47. A polygenetic tree generated, based on Dice dissimilarity matrix and cluster analysis, led to the clustering of the sunflower genotypes into two major groups (one group comprising R lines while other comprising A lines) while one genotype (CMS 234A) formed separate independent cluster. The result of dissimilarity matrix revealed high genetic diversity among all the inbreds lines. Overall, the values for genetic distances ranged from 0.13 to 0.71. The highest genetic distance (0.71) was observed between the genotypes i.e. CMS 67A and P 93R followed by 0.70 for CMS 67A and P124R. Minimum genetic distance 0.13 was observed between P 103R and P 134R, P 134R and P 145R which was indicative of common parentage of these genotypes in the pedigree tree. The principle coordinated analysis along with pattern of clustering of dissimilarity matrix separated the lines into two groups; one having B-lines and other for R-lines. The analysis of molecular variance (AMOVA) showed greater variation within genotypes (68%) as compared to between genotypes (32%).">



  <meta property="og:image" content="http://localhost:4000/images/favicon/site-treaser.webp">



  <meta name="twitter:site" content="@DrShamshadM">
  <meta name="twitter:title" content="Morphological and molecular genetic diversity analyses in Helianthus annuus (L.)">
  <meta name="twitter:description" content="The knowledge of genetic diversity among parental lines is a prerequisite for selecting parents for hybridization in any heterosis breeding programme. Evaluation of 13 parental lines using D2 analysis was undertaken during spring 2014 at PAU, Ludhiana. The genotypes were grouped into four clusters. Cluster II comprised of maximum number of genotypes (5) viz., CMS-47A, P 93R, P145R, P103R, P 167R followed by cluster I (4 genotypes i.e. CMS-11A, CMS-67A, CMS-234A, 95-C-1); cluster III and IV having two genotypes each i.e. CMS-68A, RCR 8297 and P 124R, P 134R respectively. Maximum inter cluster distance of 578.187 was recorded between genotypes of cluster II and IV while minimum inter cluster distance (175.195) was observed between genotypes present in cluster III and IV. Thirty SSR markers were used for molecular markers analysis. Total number of alleles amplified by 26 polymorphic primers was 51 with an average of 1.96 alleles per locus. The average value of polymorphic information content (PIC) for all the 26 polymorphic markers was 0.47. A polygenetic tree generated, based on Dice dissimilarity matrix and cluster analysis, led to the clustering of the sunflower genotypes into two major groups (one group comprising R lines while other comprising A lines) while one genotype (CMS 234A) formed separate independent cluster. The result of dissimilarity matrix revealed high genetic diversity among all the inbreds lines. Overall, the values for genetic distances ranged from 0.13 to 0.71. The highest genetic distance (0.71) was observed between the genotypes i.e. CMS 67A and P 93R followed by 0.70 for CMS 67A and P124R. Minimum genetic distance 0.13 was observed between P 103R and P 134R, P 134R and P 145R which was indicative of common parentage of these genotypes in the pedigree tree. The principle coordinated analysis along with pattern of clustering of dissimilarity matrix separated the lines into two groups; one having B-lines and other for R-lines. The analysis of molecular variance (AMOVA) showed greater variation within genotypes (68%) as compared to between genotypes (32%).">
  <meta name="twitter:url" content="http://localhost:4000/publication/2016-01-12-Morphological-and-Genetic-diversity-in-sunflower">

  
    <meta name="twitter:card" content="summary">
    
      <meta name="twitter:image" content="http://localhost:4000/images/favicon/site-treaser.webp">
    
  

  
    <meta name="twitter:creator" content="@DrShamshadM">
  



  <meta property="article:published_time" content="2016-01-12T00:00:00+05:30">



  <meta property="article:modified_time" content="2022-09-14T22:22:33+05:30">



  

  


<link rel="canonical" href="http://localhost:4000/publication/2016-01-12-Morphological-and-Genetic-diversity-in-sunflower">




<script async type="application/ld+json">
  {
    "@context": "https://schema.org",
    
      "@type": "Person",
      "name": "M. Shamshad",
      "url": "http://localhost:4000/"
    
  }
</script>


  <meta name="google-site-verification" content="BD1FGtpPEYJphVV41YzUn-mrOqREjhO-JsWa4YZgj60">





                                     
<!-- end _includes/seo.html -->
 
<link href="/feed.xml" type="application/atom+xml" rel="alternate" title="M. Shamshad Feed">


<!-- https://web.dev/viewport/?utm_source=lighthouse&utm_medium=lr -->
<meta name="viewport" content="width=device-width, initial-scale=1.0">

<script async>
  document.documentElement.className =
    document.documentElement.className.replace(/\bno-js\b/g, "") + " js ";
</script>

<link rel="preconnect" href="https://fonts.googleapis.com">
<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
<link href="https://fonts.googleapis.com/css2?family=PT+Serif:ital,wght@0,400;0,700;1,400;1,700&amp;display=swap" rel="stylesheet">

<!-- For all browsers -->
<link rel="stylesheet" href="/assets/css/main.css" id="theme_source">


<link rel="stylesheet alternate" href="/assets/css/mainDark.css" id="theme_source_2">
<script async>
  let theme = sessionStorage.getItem("theme");
  if (theme === "dark") {
    sessionStorage.setItem("theme", "dark");
    node1 = document.getElementById("theme_source");
    node2 = document.getElementById("theme_source_2");
    node1.setAttribute("rel", "stylesheet alternate");
    node2.setAttribute("rel", "stylesheet");
  } else {
    sessionStorage.setItem("theme", "light");
  }
</script>




    
<!-- start custom head snippets -->			   
<meta name="apple-mobile-web-app-capable" content="yes">
<meta name="apple-mobile-web-app-status-bar-style" content="default">
<link rel="apple-touch-icon" sizes="60x60" href="/images/favicon/apple-touch-icon-60x60.png?v=565802">
<link rel="apple-touch-icon" sizes="76x76" href="/images/favicon/apple-touch-icon-76x76.png?v=565802">
<link rel="apple-touch-icon" sizes="120x120" href="/images/favicon/apple-touch-icon-120x120.png?v=565802">
<link rel="apple-touch-icon" sizes="152x152" href="/images/favicon/apple-touch-icon-152x152.png?v=565802">
<link rel="apple-touch-icon" sizes="180x180" href="/images/favicon/apple-touch-icon-180x180.png?v=565802">
<link rel="icon" type="image/png" sizes="32x32" href="/images/favicon/favicon-32x32.png?v=565802">
<link rel="icon" type="image/png" sizes="16x16" href="/images/favicon/favicon-16x16.png?v=565802">
<link rel="manifest" href="/images/favicon/site.webmanifest?v=565802">
<link rel="mask-icon" href="/images/favicon/safari-pinned-tab.svg?v=565802" color="#00a7ff">
<link rel="shortcut icon" href="/images/favicon/favicon.ico?v=565802">
<meta name="apple-mobile-web-app-title" content="SM-Web">
<meta name="application-name" content="SM-Web">
<meta name="msapplication-TileColor" content="#00a300">
<meta name="msapplication-config" content="/images/favicon/browserconfig.xml?v=565802">
<meta name="theme-color" content="#ffffff">
<!-- SEO tags -->
<!-- Begin Jekyll SEO tag v2.8.0 -->
<title>Morphological and molecular genetic diversity analyses in Helianthus annuus (L.) | M. Shamshad</title>
<meta name="generator" content="Jekyll v4.3.3">
<meta property="og:title" content="Morphological and molecular genetic diversity analyses in Helianthus annuus (L.)">
<meta name="author" content="M. Shamshad">
<meta property="og:locale" content="en_US">
<meta name="description" content="The knowledge of genetic diversity among parental lines is a prerequisite for selecting parents for hybridization in any heterosis breeding programme. Evaluation of 13 parental lines using D2 analysis was undertaken during spring 2014 at PAU, Ludhiana. The genotypes were grouped into four clusters. Cluster II comprised of maximum number of genotypes (5) viz., CMS-47A, P 93R, P145R, P103R, P 167R followed by cluster I (4 genotypes i.e. CMS-11A, CMS-67A, CMS-234A, 95-C-1); cluster III and IV having two genotypes each i.e. CMS-68A, RCR 8297 and P 124R, P 134R respectively. Maximum inter cluster distance of 578.187 was recorded between genotypes of cluster II and IV while minimum inter cluster distance (175.195) was observed between genotypes present in cluster III and IV. Thirty SSR markers were used for molecular markers analysis. Total number of alleles amplified by 26 polymorphic primers was 51 with an average of 1.96 alleles per locus. The average value of polymorphic information content (PIC) for all the 26 polymorphic markers was 0.47. A polygenetic tree generated, based on Dice dissimilarity matrix and cluster analysis, led to the clustering of the sunflower genotypes into two major groups (one group comprising R lines while other comprising A lines) while one genotype (CMS 234A) formed separate independent cluster. The result of dissimilarity matrix revealed high genetic diversity among all the inbreds lines. Overall, the values for genetic distances ranged from 0.13 to 0.71. The highest genetic distance (0.71) was observed between the genotypes i.e. CMS 67A and P 93R followed by 0.70 for CMS 67A and P124R. Minimum genetic distance 0.13 was observed between P 103R and P 134R, P 134R and P 145R which was indicative of common parentage of these genotypes in the pedigree tree. The principle coordinated analysis along with pattern of clustering of dissimilarity matrix separated the lines into two groups; one having B-lines and other for R-lines. The analysis of molecular variance (AMOVA) showed greater variation within genotypes (68%) as compared to between genotypes (32%).">
<meta property="og:description" content="The knowledge of genetic diversity among parental lines is a prerequisite for selecting parents for hybridization in any heterosis breeding programme. Evaluation of 13 parental lines using D2 analysis was undertaken during spring 2014 at PAU, Ludhiana. The genotypes were grouped into four clusters. Cluster II comprised of maximum number of genotypes (5) viz., CMS-47A, P 93R, P145R, P103R, P 167R followed by cluster I (4 genotypes i.e. CMS-11A, CMS-67A, CMS-234A, 95-C-1); cluster III and IV having two genotypes each i.e. CMS-68A, RCR 8297 and P 124R, P 134R respectively. Maximum inter cluster distance of 578.187 was recorded between genotypes of cluster II and IV while minimum inter cluster distance (175.195) was observed between genotypes present in cluster III and IV. Thirty SSR markers were used for molecular markers analysis. Total number of alleles amplified by 26 polymorphic primers was 51 with an average of 1.96 alleles per locus. The average value of polymorphic information content (PIC) for all the 26 polymorphic markers was 0.47. A polygenetic tree generated, based on Dice dissimilarity matrix and cluster analysis, led to the clustering of the sunflower genotypes into two major groups (one group comprising R lines while other comprising A lines) while one genotype (CMS 234A) formed separate independent cluster. The result of dissimilarity matrix revealed high genetic diversity among all the inbreds lines. Overall, the values for genetic distances ranged from 0.13 to 0.71. The highest genetic distance (0.71) was observed between the genotypes i.e. CMS 67A and P 93R followed by 0.70 for CMS 67A and P124R. Minimum genetic distance 0.13 was observed between P 103R and P 134R, P 134R and P 145R which was indicative of common parentage of these genotypes in the pedigree tree. The principle coordinated analysis along with pattern of clustering of dissimilarity matrix separated the lines into two groups; one having B-lines and other for R-lines. The analysis of molecular variance (AMOVA) showed greater variation within genotypes (68%) as compared to between genotypes (32%).">
<link rel="canonical" href="http://localhost:4000/publication/2016-01-12-Morphological-and-Genetic-diversity-in-sunflower">
<meta property="og:url" content="http://localhost:4000/publication/2016-01-12-Morphological-and-Genetic-diversity-in-sunflower">
<meta property="og:site_name" content="M. Shamshad">
<meta property="og:type" content="article">
<meta property="article:published_time" content="2016-01-12T00:00:00+05:30">
<meta name="twitter:card" content="summary">
<meta property="twitter:title" content="Morphological and molecular genetic diversity analyses in Helianthus annuus (L.)">
<meta name="twitter:site" content="@DrShamshadM">
<meta name="twitter:creator" content="@DrShamshadM">
<meta name="google-site-verification" content="BD1FGtpPEYJphVV41YzUn-mrOqREjhO-JsWa4YZgj60">
<script type="application/ld+json">
{"@context":"https://schema.org","@type":"BlogPosting","author":{"@type":"Person","name":"M. Shamshad"},"dateModified":"2022-09-14T22:22:33+05:30","datePublished":"2016-01-12T00:00:00+05:30","description":"The knowledge of genetic diversity among parental lines is a prerequisite for selecting parents for hybridization in any heterosis breeding programme. Evaluation of 13 parental lines using D2 analysis was undertaken during spring 2014 at PAU, Ludhiana. The genotypes were grouped into four clusters. Cluster II comprised of maximum number of genotypes (5) viz., CMS-47A, P 93R, P145R, P103R, P 167R followed by cluster I (4 genotypes i.e. CMS-11A, CMS-67A, CMS-234A, 95-C-1); cluster III and IV having two genotypes each i.e. CMS-68A, RCR 8297 and P 124R, P 134R respectively. Maximum inter cluster distance of 578.187 was recorded between genotypes of cluster II and IV while minimum inter cluster distance (175.195) was observed between genotypes present in cluster III and IV. Thirty SSR markers were used for molecular markers analysis. Total number of alleles amplified by 26 polymorphic primers was 51 with an average of 1.96 alleles per locus. The average value of polymorphic information content (PIC) for all the 26 polymorphic markers was 0.47. A polygenetic tree generated, based on Dice dissimilarity matrix and cluster analysis, led to the clustering of the sunflower genotypes into two major groups (one group comprising R lines while other comprising A lines) while one genotype (CMS 234A) formed separate independent cluster. The result of dissimilarity matrix revealed high genetic diversity among all the inbreds lines. Overall, the values for genetic distances ranged from 0.13 to 0.71. The highest genetic distance (0.71) was observed between the genotypes i.e. CMS 67A and P 93R followed by 0.70 for CMS 67A and P124R. Minimum genetic distance 0.13 was observed between P 103R and P 134R, P 134R and P 145R which was indicative of common parentage of these genotypes in the pedigree tree. The principle coordinated analysis along with pattern of clustering of dissimilarity matrix separated the lines into two groups; one having B-lines and other for R-lines. The analysis of molecular variance (AMOVA) showed greater variation within genotypes (68%) as compared to between genotypes (32%).","headline":"Morphological and molecular genetic diversity analyses in Helianthus annuus (L.)","mainEntityOfPage":{"@type":"WebPage","@id":"http://localhost:4000/publication/2016-01-12-Morphological-and-Genetic-diversity-in-sunflower"},"url":"http://localhost:4000/publication/2016-01-12-Morphological-and-Genetic-diversity-in-sunflower"}</script>
<!-- End Jekyll SEO tag -->

<!-- end custom head snippets -->
  </head>

  <body class="layout--single">
    <nav class="skip-links">
  <ul>
    <li><a href="#site-nav" class="screen-reader-shortcut">Skip to primary navigation</a></li>
    <li><a href="#main" class="screen-reader-shortcut">Skip to content</a></li>
    <li><a href="#footer" class="screen-reader-shortcut">Skip to footer</a></li>
  </ul>
</nav>

    

<div class="masthead">
  <div class="masthead__inner-wrap">
    <div class="masthead__menu">
      <nav id="site-nav" class="greedy-nav">
        
        <a class="site-title" href="/">
          M. Shamshad
          
        </a>
        <ul class="visible-links">
<li class="masthead__menu-item">
              <a href="/about/">About</a>
            </li>
<li class="masthead__menu-item">
              <a href="/teaching/">Teaching</a>
            </li>
<li class="masthead__menu-item">
              <a href="/publications/">Publications</a>
            </li>
<li class="masthead__menu-item">
              <a href="/talks/">Talks</a>
            </li>
<li class="masthead__menu-item">
              <a href="/research/">Research</a>
            </li>
<li class="masthead__menu-item">
              <a href="/resume/">Resume</a>
            </li>
<li class="masthead__menu-item">
              <a href="/tags/">Tags</a>
            </li>
</ul>
         
        <span class="fa-solid fa-moon toggle-switch" style="cursor: pointer;" aria-hidden="true" onclick="node1=document.getElementById('theme_source');node2=document.getElementById('theme_source_2');if(node1.getAttribute('rel')=='stylesheet'){node1.setAttribute('rel', 'stylesheet alternate'); node2.setAttribute('rel', 'stylesheet');sessionStorage.setItem('theme', 'dark');}else{node2.setAttribute('rel', 'stylesheet alternate'); node1.setAttribute('rel', 'stylesheet');sessionStorage.setItem('theme', 'light');} return false;"></span>
        
        
        
        <button class="greedy-nav__toggle hidden" type="button">
          <span class="visually-hidden">Toggle menu</span>
          <div class="navicon"></div>
        </button>
        <ul class="hidden-links hidden"></ul>
      </nav>
    </div>
  </div>
</div>


    <div class="initial-content">
      



<div id="main" role="main">
  
  <div class="sidebar sticky">
  




<div itemscope itemtype="https://schema.org/Person" class="h-card">

  
  <div class="author__avatar">
   <a href="http://localhost:4000/">
    	<img src="/images/profile.webp" alt="M. Shamshad" itemprop="image" class="u-photo">
   </a>
  </div>
  

  <div class="author__content">
      <h3 class="author__name p-name" itemprop="name">
      <a class="u-url" rel="me" href="http://localhost:4000/" itemprop="url">M. Shamshad</a>
    </h3>
	   
      <div class="author__bio p-note" itemprop="description">
        <p>Plant Breeding &amp;&amp; Data Science</p>

      </div>
       
  </div>

  <div class="author__urls-wrapper">
    <button class="btn btn--inverse">Follow</button>
    <ul class="author__urls social-icons">
      
              <li itemprop="homeLocation" itemscope itemtype="https://schema.org/Place">
          <i class="fas fa-fw fa-map-marker-alt" aria-hidden="true"></i> <span itemprop="name" class="p-locality">Ludhina, Punjab, India</span>
        </li>     
      

      

      

      
        <li>
          <a href="mailto:shamshad.rattan@gmail.com" rel="me" class="u-email">
            <meta itemprop="email" content="shamshad.rattan@gmail.com">
            <i class="fas fa-fw fa-envelope" aria-hidden="true"></i><span class="label">Email</span>
          </a>
        </li>
      

      
       
        <li><a href="https://www.researchgate.net/profile/Mohd-Shamshad"><i class="ai  ai-researchgate-square ai-fw" aria-hidden="true"></i> ResearchGate</a></li>
      

      
        <li>
          <a href="https://twitter.com/DrShamshadM" itemprop="sameAs" rel="nofollow noopener noreferrer me">
            <i class="fab fa-fw fa-twitter-square" aria-hidden="true"></i><span class="label">Twitter</span>
          </a>
        </li>
      

      

      
        <li>
          <a href="https://www.linkedin.com/in/dr-m-shamshad-58b4321b1" itemprop="sameAs" rel="nofollow noopener noreferrer me">
            <i class="fab fa-fw fa-linkedin" aria-hidden="true"></i><span class="label">Linkedin</span>
          </a>
        </li>
      

      

      

      

      

      

      

      

      

      

      

      

      

      

      

      

      

      

       
      
      <li><a href="https://scholar.google.com/citations?user=dI7V1a4AAAAJ&hl=en" itemprop="sameAs" rel="nofollow noopener noreferrer me">
        <i class="ai ai-fw ai-google-scholar" aria-hidden="true"></i><span class="label">Google Scholar</span></a></li>
                 
      
      
                                                                                                   
       
          <!--
  <li>
    <a href="https://link-to-whatever-social-network.com/user/" itemprop="sameAs" rel="nofollow noopener noreferrer">
      <i class="fas fa-fw" aria-hidden="true"></i> Custom Social Profile Link
    </a>
  </li>
-->
    </ul>
  </div>
</div>

  
  </div>


  <article class="page" itemscope itemtype="https://schema.org/CreativeWork">
    <meta itemprop="headline" content="Morphological and molecular genetic diversity analyses in Helianthus annuus (L.)">
    <meta itemprop="description" content="The knowledge of genetic diversity among parental lines is a prerequisite for selecting parents for hybridization in any heterosis breeding programme. Evaluation of 13 parental lines using D2 analysis was undertaken during spring 2014 at PAU, Ludhiana. The genotypes were grouped into four clusters. Cluster II comprised of maximum number of genotypes (5) viz., CMS-47A, P 93R, P145R, P103R, P 167R followed by cluster I (4 genotypes i.e. CMS-11A, CMS-67A, CMS-234A, 95-C-1); cluster III and IV having two genotypes each i.e. CMS-68A, RCR 8297 and P 124R, P 134R respectively. Maximum inter cluster distance of 578.187 was recorded between genotypes of cluster II and IV while minimum inter cluster distance (175.195) was observed between genotypes present in cluster III and IV. Thirty SSR markers were used for molecular markers analysis. Total number of alleles amplified by 26 polymorphic primers was 51 with an average of 1.96 alleles per locus. The average value of polymorphic information content (PIC) for all the 26 polymorphic markers was 0.47. A polygenetic tree generated, based on Dice dissimilarity matrix and cluster analysis, led to the clustering of the sunflower genotypes into two major groups (one group comprising R lines while other comprising A lines) while one genotype (CMS 234A) formed separate independent cluster. The result of dissimilarity matrix revealed high genetic diversity among all the inbreds lines. Overall, the values for genetic distances ranged from 0.13 to 0.71. The highest genetic distance (0.71) was observed between the genotypes i.e. CMS 67A and P 93R followed by 0.70 for CMS 67A and P124R. Minimum genetic distance 0.13 was observed between P 103R and P 134R, P 134R and P 145R which was indicative of common parentage of these genotypes in the pedigree tree. The principle coordinated analysis along with pattern of clustering of dissimilarity matrix separated the lines into two groups; one having B-lines and other for R-lines. The analysis of molecular variance (AMOVA) showed greater variation within genotypes (68%) as compared to between genotypes (32%).">
    <meta itemprop="datePublished" content="2016-01-12T00:00:00+05:30">
    <meta itemprop="dateModified" content="2022-09-14T22:22:33+05:30">

    <div class="page__inner-wrap">
      
        <header>
          <h1 id="page-title" class="page__title" itemprop="headline">Morphological and molecular genetic diversity analyses in <i>Helianthus annuus</i> (L.)
</h1>
		 
        
        
        
          <p>Published in <i>Electronic Journal of Plant Breeding</i>, 2016 </p>
        
        
             
        <!-- 
          <p>Recommended citation: <b>Shamshad, M.</b> &amp; Dhillon, S. K. (2016). &quot;Morphological and molecular genetic diversity analyses in <i>Helianthus annuus</i> (L.)&quot;. <i>Electronic Journal of Plant Breeding</i>, 7(4), 1216-1223. Retrieved from https://doi.org/10.5958/0975-928X.2016.00169 </p>
         -->
        </header>
      

      <section class="page__content" itemprop="text">
        
        <p><a href="/files/research/Morphological.pdf">Download PDF here</a></p>

<p>Abstract: The knowledge of genetic diversity among parental lines is a prerequisite for selecting parents for hybridization in any heterosis breeding programme. Evaluation of 13 parental lines using D2 analysis was undertaken during spring 2014 at PAU, Ludhiana. The genotypes were grouped into four clusters. Cluster II comprised of maximum number of genotypes (5) viz., CMS-47A, P 93R, P145R, P103R, P 167R followed by cluster I (4 genotypes i.e. CMS-11A, CMS-67A, CMS-234A, 95-C-1); cluster III and IV having two genotypes each i.e. CMS-68A, RCR 8297 and P 124R, P 134R respectively. Maximum inter cluster distance of 578.187 was recorded between genotypes of cluster II and IV while minimum inter cluster distance (175.195) was observed between genotypes present in cluster III and IV. Thirty SSR markers were used for molecular markers analysis. Total number of alleles amplified by 26 polymorphic primers was 51 with an average of 1.96 alleles per locus. The average value of polymorphic information content (PIC) for all the 26 polymorphic markers was 0.47. A polygenetic tree generated, based on Dice dissimilarity matrix and cluster analysis, led to the clustering of the sunflower genotypes into two major groups (one group comprising R lines while other comprising A lines) while one genotype (CMS 234A) formed separate independent cluster. The result of dissimilarity matrix revealed high genetic diversity among all the inbreds lines. Overall, the values for genetic distances ranged from 0.13 to 0.71. The highest genetic distance (0.71) was observed between the genotypes i.e. CMS 67A and P 93R followed by 0.70 for CMS 67A and P124R. Minimum genetic distance 0.13 was observed between P 103R and P 134R, P 134R and P 145R which was indicative of common parentage of these genotypes in the pedigree tree. The principle coordinated analysis along with pattern of clustering of dissimilarity matrix separated the lines into two groups; one having B-lines and other for R-lines. The analysis of molecular variance (AMOVA) showed greater variation within genotypes (68%) as compared to between genotypes (32%).</p>

<p>Recommended citation: <b>Shamshad, M.</b> &amp; Dhillon, S. K. (2016). “Morphological and molecular genetic diversity analyses in <i>Helianthus annuus</i> (L.)”. <i>Electronic Journal of Plant Breeding</i>, 7(4), 1216-1223. Retrieved from https://doi.org/10.5958/0975-928X.2016.00169</p>

        
      </section>

      <footer class="page__meta">
        
        




        

  <p class="page__date"><strong><i class="fas fa-fw fa-calendar-alt" aria-hidden="true"></i> Updated:</strong> <time datetime="2022-09-14">September 14, 2022</time></p>


      </footer>

      

<section class="page__share">
  
    <h4 class="page__share-title">Share on</h4>
  

  <a href="https://twitter.com/intent/tweet?via=DrShamshadM&text=Morphological+and+molecular+genetic+diversity+analyses+in+%3Ci%3EHelianthus+annuus%3C%2Fi%3E+%28L.%29%20http%3A%2F%2Flocalhost%3A4000%2Fpublication%2F2016-01-12-Morphological-and-Genetic-diversity-in-sunflower" class="btn btn--twitter" onclick="window.open(this.href, 'window', 'left=20,top=20,width=500,height=500,toolbar=1,resizable=0'); return false;" title="Share on Twitter"><i class="fab fa-fw fa-twitter" aria-hidden="true"></i><span> Twitter</span></a>

  <a href="https://www.facebook.com/sharer/sharer.php?u=http%3A%2F%2Flocalhost%3A4000%2Fpublication%2F2016-01-12-Morphological-and-Genetic-diversity-in-sunflower" class="btn btn--facebook" onclick="window.open(this.href, 'window', 'left=20,top=20,width=500,height=500,toolbar=1,resizable=0'); return false;" title="Share on Facebook"><i class="fab fa-fw fa-facebook" aria-hidden="true"></i><span> Facebook</span></a>

  <a href="https://www.linkedin.com/shareArticle?mini=true&url=http%3A%2F%2Flocalhost%3A4000%2Fpublication%2F2016-01-12-Morphological-and-Genetic-diversity-in-sunflower" class="btn btn--linkedin" onclick="window.open(this.href, 'window', 'left=20,top=20,width=500,height=500,toolbar=1,resizable=0'); return false;" title="Share on LinkedIn"><i class="fab fa-fw fa-linkedin" aria-hidden="true"></i><span> LinkedIn</span></a>
</section>

      


  <nav class="pagination">
    
      <a href="http://localhost:4000/publication/2014-07-11-Combinig-Ability-sunflower" class="pagination--pager" title="Magnitude of combining ability for seed yield and oil content in sunflower (Helianthus annuus L.)
">Previous</a>
    
    
      <a href="http://localhost:4000/publication/2016-05-18-Heterosis-for-oil-and-oil-content" class="pagination--pager" title="Heterosis for oil content and oil quality in sunflower (Helianthus annuus L.)
">Next</a>
    
  </nav>

    </div>

    
  </article>

  
  
</div>

    </div>

    

    <div id="footer" class="page__footer">
      <footer>
        <!-- start custom footer snippets -->

<!-- end custom footer snippets -->
        <div class="page__footer-follow">
  <ul class="social-icons">
    
      <li><strong>Follow:</strong></li>
    

    
      
        
          <li><a href="https://twitter.com/DrShamshadM" rel="nofollow noopener noreferrer"><i class="fab fa-fw fa-twitter-square" aria-hidden="true"></i> Twitter</a></li>
        
      
        
          <li><a href="https://scholar.google.com/citations?user=dI7V1a4AAAAJ&hl=en" rel="nofollow noopener noreferrer"><i class="ai ai-google-scholar-square ai-fw" aria-hidden="true"></i> Google Scholar</a></li>
        
      
        
          <li><a href="https://www.researchgate.net/profile/Mohd-Shamshad" rel="nofollow noopener noreferrer"><i class="ai ai-researchgate-square ai-fw" aria-hidden="true"></i> ResearchGate</a></li>
        
      
    

    
      <li><a href="/feed.xml"><i class="fas fa-fw fa-rss-square" aria-hidden="true"></i> Feed</a></li>
    
  </ul>
</div>

<div class="page__footer-copyright">© 2024 M. Shamshad. Proudly hosted by <a href="https://github.com/" rel="nofollow">GitHub</a> &amp; <a href="https://shamshadpbg.github.io/" rel="prev:"><i class="ai ai-academia"></i> AyeShmMz</a>.</div>

      </footer>
    </div>

    
  <script src="/assets/js/main.min.js" defer></script>
  <!-- <script src="https://kit.fontawesome.com/4eee35f757.js"></script> -->












  </body>
</html>
