<!doctype html><html lang="en-US" class="no-js"><head><meta charset="utf-8"> <!-- begin _includes/seo.html --><title>RNA-Seq-Based Transcriptomics Study to Investigate the Genes Governing Nitrogen Use Efficiency in Indian Wheat Cultivars | M. Shamshad</title><meta name="description" content="High NUE (nitrogen use efficiency) has great practical significance for sustainable crop production. Wheat is one of the main cultivated crops worldwide for human food and nutrition. However, wheat grain productivity is dependent upon cultivars with high NUE in addition to the application of nitrogen fertilizers. In order to understand the molecular mechanisms exhibiting a high NUE response, a comparative transcriptomics study was carried out through RNA-seq analysis to investigate the gene expression that regulates NUE, in root and shoot tissue of N-efficient (PBW677) and N-inefficient (703) cultivars under optimum and nitrogen (N) stress. Differentially expressed gene analysis revealed a total of 2,406 differentially expressed genes (DEGs) present in both the contrasting cultivars under N stress. The efficient genotype PBW677 had considerably more abundant DEGs with 1,653 (903 roots +750 shoots) compared to inefficient cultivar PBW703 with 753 (96 roots +657 shoots). Gene ontology enrichment and pathway analysis of these DEGs suggested that the two cultivars differed in terms of adaptive mechanism. Gene enrichment analysis revealed that among the upregulated and downregulated genes the overrepresented and underrepresented gene categories belonged to biological processes like DNA binding, response to abiotic stimulus, photosynthesis, carbon fixation, carbohydrate metabolic process, nitrogen compound metabolic process, nitrate transport, and translation in cultivar PBW677, while the enriched biological processes were nucleosome assembly, chromatin remodeling, DNA packaging, lipid transport, sulfur compound metabolic process, protein modifications, and protein folding and refolding in N inefficient cultivar PBW703. We found several transcription factors (MYB, WRKY, RING finger protein, zinc finger protein, transporters, NRT1, amino acid transporters, sugar), protein kinases, and genes involved in N absorption, transportation, and assimilation to be highly expressed in high NUE cultivar PBW677. In our study, we report 13 potential candidate genes which showed alternate gene expression in the two contrasting cultivars under study. These genes could serve as potential targets for future breeding programs."><meta name="author" content="M. Shamshad"><meta property="article:author" content="M. Shamshad"><meta property="og:type" content="article"><meta property="og:locale" content="en_US"><meta property="og:site_name" content="M. Shamshad"><meta property="og:title" content="RNA-Seq-Based Transcriptomics Study to Investigate the Genes Governing Nitrogen Use Efficiency in Indian Wheat Cultivars"><meta property="og:url" content="https://shamshad.in/publication/2022-03-31-RNA-Seq-NUE,-differential-gene-expression"><meta property="og:description" content="High NUE (nitrogen use efficiency) has great practical significance for sustainable crop production. Wheat is one of the main cultivated crops worldwide for human food and nutrition. However, wheat grain productivity is dependent upon cultivars with high NUE in addition to the application of nitrogen fertilizers. In order to understand the molecular mechanisms exhibiting a high NUE response, a comparative transcriptomics study was carried out through RNA-seq analysis to investigate the gene expression that regulates NUE, in root and shoot tissue of N-efficient (PBW677) and N-inefficient (703) cultivars under optimum and nitrogen (N) stress. Differentially expressed gene analysis revealed a total of 2,406 differentially expressed genes (DEGs) present in both the contrasting cultivars under N stress. The efficient genotype PBW677 had considerably more abundant DEGs with 1,653 (903 roots +750 shoots) compared to inefficient cultivar PBW703 with 753 (96 roots +657 shoots). Gene ontology enrichment and pathway analysis of these DEGs suggested that the two cultivars differed in terms of adaptive mechanism. Gene enrichment analysis revealed that among the upregulated and downregulated genes the overrepresented and underrepresented gene categories belonged to biological processes like DNA binding, response to abiotic stimulus, photosynthesis, carbon fixation, carbohydrate metabolic process, nitrogen compound metabolic process, nitrate transport, and translation in cultivar PBW677, while the enriched biological processes were nucleosome assembly, chromatin remodeling, DNA packaging, lipid transport, sulfur compound metabolic process, protein modifications, and protein folding and refolding in N inefficient cultivar PBW703. We found several transcription factors (MYB, WRKY, RING finger protein, zinc finger protein, transporters, NRT1, amino acid transporters, sugar), protein kinases, and genes involved in N absorption, transportation, and assimilation to be highly expressed in high NUE cultivar PBW677. In our study, we report 13 potential candidate genes which showed alternate gene expression in the two contrasting cultivars under study. These genes could serve as potential targets for future breeding programs."><meta property="og:image" content="https://shamshad.in/images/favicon/site-treaser.webp"><meta name="twitter:site" content="@DrShamshadM"><meta name="twitter:title" content="RNA-Seq-Based Transcriptomics Study to Investigate the Genes Governing Nitrogen Use Efficiency in Indian Wheat Cultivars"><meta name="twitter:description" content="High NUE (nitrogen use efficiency) has great practical significance for sustainable crop production. Wheat is one of the main cultivated crops worldwide for human food and nutrition. However, wheat grain productivity is dependent upon cultivars with high NUE in addition to the application of nitrogen fertilizers. In order to understand the molecular mechanisms exhibiting a high NUE response, a comparative transcriptomics study was carried out through RNA-seq analysis to investigate the gene expression that regulates NUE, in root and shoot tissue of N-efficient (PBW677) and N-inefficient (703) cultivars under optimum and nitrogen (N) stress. Differentially expressed gene analysis revealed a total of 2,406 differentially expressed genes (DEGs) present in both the contrasting cultivars under N stress. The efficient genotype PBW677 had considerably more abundant DEGs with 1,653 (903 roots +750 shoots) compared to inefficient cultivar PBW703 with 753 (96 roots +657 shoots). Gene ontology enrichment and pathway analysis of these DEGs suggested that the two cultivars differed in terms of adaptive mechanism. Gene enrichment analysis revealed that among the upregulated and downregulated genes the overrepresented and underrepresented gene categories belonged to biological processes like DNA binding, response to abiotic stimulus, photosynthesis, carbon fixation, carbohydrate metabolic process, nitrogen compound metabolic process, nitrate transport, and translation in cultivar PBW677, while the enriched biological processes were nucleosome assembly, chromatin remodeling, DNA packaging, lipid transport, sulfur compound metabolic process, protein modifications, and protein folding and refolding in N inefficient cultivar PBW703. We found several transcription factors (MYB, WRKY, RING finger protein, zinc finger protein, transporters, NRT1, amino acid transporters, sugar), protein kinases, and genes involved in N absorption, transportation, and assimilation to be highly expressed in high NUE cultivar PBW677. In our study, we report 13 potential candidate genes which showed alternate gene expression in the two contrasting cultivars under study. These genes could serve as potential targets for future breeding programs."><meta name="twitter:url" content="https://shamshad.in/publication/2022-03-31-RNA-Seq-NUE,-differential-gene-expression"><meta name="twitter:card" content="summary"><meta name="twitter:image" content="https://shamshad.in/images/favicon/site-treaser.webp"><meta name="twitter:creator" content="@DrShamshadM"><meta property="article:published_time" content="2022-03-31T00:00:00+05:30"><meta property="article:modified_time" content="2024-10-06T20:53:46+05:30"><link rel="canonical" href="https://shamshad.in/publication/2022-03-31-RNA-Seq-NUE,-differential-gene-expression"> <!-- end _includes/seo.html --><link href="/feed.xml" type="application/atom+xml" rel="alternate" title="M. Shamshad Feed"> <!-- https://web.dev/viewport/?utm_source=lighthouse&utm_medium=lr --><meta name="viewport" content="width=device-width, initial-scale=1.0"> <script defer> document.documentElement.className = document.documentElement.className.replace(/\bno-js\b/g, '') + ' js '; window.enable_copy_code_button = true; </script><link rel="stylesheet" href="/assets/academicons-1.9.4/css/academicons.min.css" fetchpriority="low"/> <!-- For all browsers --><link rel="stylesheet" href="/assets/css/main.css" id="theme_source" /><link rel="stylesheet alternate" href="/assets/css/mainDark.css" id="theme_source_2" /> <script defer> let theme = sessionStorage.getItem("theme"); if (theme === "dark" || window.matchMedia('(prefers-color-scheme: dark)').matches) { sessionStorage.setItem("theme", "dark"); node1 = document.getElementById("theme_source"); node2 = document.getElementById("theme_source_2"); node1.setAttribute("rel", "stylesheet alternate"); node2.setAttribute("rel", "stylesheet"); } else { sessionStorage.setItem("theme", "light"); } </script> <!-- start custom head snippets --><meta name="apple-mobile-web-app-capable" content="yes" /><meta name="apple-mobile-web-app-status-bar-style" content="default" /><link rel="apple-touch-icon" sizes="60x60" href="/images/favicon/apple-touch-icon-60x60.png?v=565802"><link rel="apple-touch-icon" sizes="76x76" href="/images/favicon/apple-touch-icon-76x76.png?v=565802"><link rel="apple-touch-icon" sizes="120x120" href="/images/favicon/apple-touch-icon-120x120.png?v=565802"><link rel="apple-touch-icon" sizes="152x152" href="/images/favicon/apple-touch-icon-152x152.png?v=565802"><link rel="apple-touch-icon" sizes="180x180" href="/images/favicon/apple-touch-icon-180x180.png?v=565802"><link rel="icon" type="image/png" sizes="32x32" href="/images/favicon/favicon-32x32.png?v=565802"><link rel="icon" type="image/png" sizes="16x16" href="/images/favicon/favicon-16x16.png?v=565802"><link rel="manifest" href="/images/favicon/site.webmanifest?v=565802"><link rel="mask-icon" href="/images/favicon/safari-pinned-tab.svg?v=565802" color="#00a7ff"><link rel="shortcut icon" href="/images/favicon/favicon.ico?v=565802"><meta name="apple-mobile-web-app-title" content="SM-Web"><meta name="application-name" content="SM-Web"><meta name="msapplication-TileColor" content="#00a300"><meta name="msapplication-config" content="/images/favicon/browserconfig.xml?v=565802"><meta name="theme-color" content="#ffffff"> <!-- SEO tags --> <!-- Begin Jekyll SEO tag v2.8.0 --><title>RNA-Seq-Based Transcriptomics Study to Investigate the Genes Governing Nitrogen Use Efficiency in Indian Wheat Cultivars | M. Shamshad</title><meta name="generator" content="Jekyll v3.10.0" /><meta property="og:title" content="RNA-Seq-Based Transcriptomics Study to Investigate the Genes Governing Nitrogen Use Efficiency in Indian Wheat Cultivars" /><meta name="author" content="M. Shamshad" /><meta property="og:locale" content="en_US" /><meta name="description" content="High NUE (nitrogen use efficiency) has great practical significance for sustainable crop production. Wheat is one of the main cultivated crops worldwide for human food and nutrition. However, wheat grain productivity is dependent upon cultivars with high NUE in addition to the application of nitrogen fertilizers. In order to understand the molecular mechanisms exhibiting a high NUE response, a comparative transcriptomics study was carried out through RNA-seq analysis to investigate the gene expression that regulates NUE, in root and shoot tissue of N-efficient (PBW677) and N-inefficient (703) cultivars under optimum and nitrogen (N) stress. Differentially expressed gene analysis revealed a total of 2,406 differentially expressed genes (DEGs) present in both the contrasting cultivars under N stress. The efficient genotype PBW677 had considerably more abundant DEGs with 1,653 (903 roots +750 shoots) compared to inefficient cultivar PBW703 with 753 (96 roots +657 shoots). Gene ontology enrichment and pathway analysis of these DEGs suggested that the two cultivars differed in terms of adaptive mechanism. Gene enrichment analysis revealed that among the upregulated and downregulated genes the overrepresented and underrepresented gene categories belonged to biological processes like DNA binding, response to abiotic stimulus, photosynthesis, carbon fixation, carbohydrate metabolic process, nitrogen compound metabolic process, nitrate transport, and translation in cultivar PBW677, while the enriched biological processes were nucleosome assembly, chromatin remodeling, DNA packaging, lipid transport, sulfur compound metabolic process, protein modifications, and protein folding and refolding in N inefficient cultivar PBW703. We found several transcription factors (MYB, WRKY, RING finger protein, zinc finger protein, transporters, NRT1, amino acid transporters, sugar), protein kinases, and genes involved in N absorption, transportation, and assimilation to be highly expressed in high NUE cultivar PBW677. In our study, we report 13 potential candidate genes which showed alternate gene expression in the two contrasting cultivars under study. These genes could serve as potential targets for future breeding programs." /><meta property="og:description" content="High NUE (nitrogen use efficiency) has great practical significance for sustainable crop production. Wheat is one of the main cultivated crops worldwide for human food and nutrition. However, wheat grain productivity is dependent upon cultivars with high NUE in addition to the application of nitrogen fertilizers. In order to understand the molecular mechanisms exhibiting a high NUE response, a comparative transcriptomics study was carried out through RNA-seq analysis to investigate the gene expression that regulates NUE, in root and shoot tissue of N-efficient (PBW677) and N-inefficient (703) cultivars under optimum and nitrogen (N) stress. Differentially expressed gene analysis revealed a total of 2,406 differentially expressed genes (DEGs) present in both the contrasting cultivars under N stress. The efficient genotype PBW677 had considerably more abundant DEGs with 1,653 (903 roots +750 shoots) compared to inefficient cultivar PBW703 with 753 (96 roots +657 shoots). Gene ontology enrichment and pathway analysis of these DEGs suggested that the two cultivars differed in terms of adaptive mechanism. Gene enrichment analysis revealed that among the upregulated and downregulated genes the overrepresented and underrepresented gene categories belonged to biological processes like DNA binding, response to abiotic stimulus, photosynthesis, carbon fixation, carbohydrate metabolic process, nitrogen compound metabolic process, nitrate transport, and translation in cultivar PBW677, while the enriched biological processes were nucleosome assembly, chromatin remodeling, DNA packaging, lipid transport, sulfur compound metabolic process, protein modifications, and protein folding and refolding in N inefficient cultivar PBW703. We found several transcription factors (MYB, WRKY, RING finger protein, zinc finger protein, transporters, NRT1, amino acid transporters, sugar), protein kinases, and genes involved in N absorption, transportation, and assimilation to be highly expressed in high NUE cultivar PBW677. In our study, we report 13 potential candidate genes which showed alternate gene expression in the two contrasting cultivars under study. These genes could serve as potential targets for future breeding programs." /><link rel="canonical" href="https://shamshad.in/publication/2022-03-31-RNA-Seq-NUE,-differential-gene-expression" /><meta property="og:url" content="https://shamshad.in/publication/2022-03-31-RNA-Seq-NUE,-differential-gene-expression" /><meta property="og:site_name" content="M. Shamshad" /><meta property="og:type" content="article" /><meta property="article:published_time" content="2022-03-31T00:00:00+05:30" /><meta name="twitter:card" content="summary" /><meta property="twitter:title" content="RNA-Seq-Based Transcriptomics Study to Investigate the Genes Governing Nitrogen Use Efficiency in Indian Wheat Cultivars" /><meta name="twitter:site" content="@DrShamshadM" /><meta name="twitter:creator" content="@DrShamshadM" /> <script type="application/ld+json"> {"@context":"https://schema.org","@type":"BlogPosting","author":{"@type":"Person","name":"M. Shamshad"},"dateModified":"2024-10-06T20:53:46+05:30","datePublished":"2022-03-31T00:00:00+05:30","description":"High NUE (nitrogen use efficiency) has great practical significance for sustainable crop production. Wheat is one of the main cultivated crops worldwide for human food and nutrition. However, wheat grain productivity is dependent upon cultivars with high NUE in addition to the application of nitrogen fertilizers. In order to understand the molecular mechanisms exhibiting a high NUE response, a comparative transcriptomics study was carried out through RNA-seq analysis to investigate the gene expression that regulates NUE, in root and shoot tissue of N-efficient (PBW677) and N-inefficient (703) cultivars under optimum and nitrogen (N) stress. Differentially expressed gene analysis revealed a total of 2,406 differentially expressed genes (DEGs) present in both the contrasting cultivars under N stress. The efficient genotype PBW677 had considerably more abundant DEGs with 1,653 (903 roots +750 shoots) compared to inefficient cultivar PBW703 with 753 (96 roots +657 shoots). Gene ontology enrichment and pathway analysis of these DEGs suggested that the two cultivars differed in terms of adaptive mechanism. Gene enrichment analysis revealed that among the upregulated and downregulated genes the overrepresented and underrepresented gene categories belonged to biological processes like DNA binding, response to abiotic stimulus, photosynthesis, carbon fixation, carbohydrate metabolic process, nitrogen compound metabolic process, nitrate transport, and translation in cultivar PBW677, while the enriched biological processes were nucleosome assembly, chromatin remodeling, DNA packaging, lipid transport, sulfur compound metabolic process, protein modifications, and protein folding and refolding in N inefficient cultivar PBW703. We found several transcription factors (MYB, WRKY, RING finger protein, zinc finger protein, transporters, NRT1, amino acid transporters, sugar), protein kinases, and genes involved in N absorption, transportation, and assimilation to be highly expressed in high NUE cultivar PBW677. In our study, we report 13 potential candidate genes which showed alternate gene expression in the two contrasting cultivars under study. These genes could serve as potential targets for future breeding programs.","headline":"RNA-Seq-Based Transcriptomics Study to Investigate the Genes Governing Nitrogen Use Efficiency in Indian Wheat Cultivars","mainEntityOfPage":{"@type":"WebPage","@id":"https://shamshad.in/publication/2022-03-31-RNA-Seq-NUE,-differential-gene-expression"},"publisher":{"@type":"Organization","logo":{"@type":"ImageObject","url":"https://shamshad.in/images/favicon/site-treaser.webp"},"name":"M. Shamshad"},"url":"https://shamshad.in/publication/2022-03-31-RNA-Seq-NUE,-differential-gene-expression"}</script> <!-- End Jekyll SEO tag --></head><body class="layout--single" dir="ltr"><nav class="skip-links"><ul>
<li><a href="#site-nav" class="screen-reader-shortcut">Skip to primary navigation</a></li>
<li><a href="#main" class="screen-reader-shortcut">Skip to content</a></li>
<li><a href="#footer" class="screen-reader-shortcut">Skip to footer</a></li>
</ul></nav><div class="masthead"><div class="masthead__inner-wrap"><div class="masthead__menu"><nav id="site-nav" class="greedy-nav"> <a class="site-logo" href="/"><img src="/images/favicon/site-treaser.webp" alt="M. Shamshad"></a> <a class="site-title" href="/"> M. Shamshad </a><ul class="visible-links">
<li class="masthead__menu-item"> <a href="/about/">About</a>
</li>
<li class="masthead__menu-item"> <a href="/teaching/">Teaching</a>
</li>
<li class="masthead__menu-item"> <a href="/publications/">Publications</a>
</li>
<li class="masthead__menu-item"> <a href="/talks/">Talks</a>
</li>
<li class="masthead__menu-item"> <a href="/research/">Research</a>
</li>
<li class="masthead__menu-item"> <a href="/resume/">Resume</a>
</li>
<li class="masthead__menu-item"> <a href="/tags/">Tags</a>
</li>
<li class="masthead__menu-item"> <a href="/docs/Linux_tutorial/">Bioinfo</a>
</li>
</ul>
<span class="fa-solid fa-moon toggle-switch" type="button" style="cursor: pointer;" aria-hidden="true" onclick="node1 = document.getElementById('theme_source'); node2 = document.getElementById('theme_source_2'); if(node1.getAttribute('rel')=='stylesheet'){node1.setAttribute('rel', 'stylesheet alternate'); node2.setAttribute('rel', 'stylesheet'); sessionStorage.setItem('theme', 'dark');} else{node2.setAttribute('rel', 'stylesheet alternate'); node1.setAttribute('rel', 'stylesheet');sessionStorage.setItem('theme', 'light');} return false;"></span> <button class="greedy-nav__toggle hidden" type="button"> <span class="visually-hidden">Toggle menu</span><div class="navicon"></div></button><ul class="hidden-links hidden"></ul></nav></div></div></div><div class="initial-content"><div id="main" role="main">
<div class="sidebar sticky"><div itemscope itemtype="https://schema.org/Person" class="h-card">
<div class="author__avatar"> <a href="https://shamshad.in/"> <img loading="lazy" src="/images/profile.webp" alt="M. Shamshad" itemprop="image" class="u-photo"> </a>
</div>
<div class="author__content">
<h3 class="author__name p-name" itemprop="name"> <a class="u-url" rel="me" href="https://shamshad.in/" itemprop="url">M. Shamshad</a>
</h3>
<div class="author__bio p-note" itemprop="description"><p>Plant Breeding &amp;&amp; Data Science</p></div>
</div>
<div class="author__urls-wrapper"> <button class="btn btn--inverse">Follow</button><ul class="author__urls social-icons">
<li itemprop="homeLocation" itemscope itemtype="https://schema.org/Place"> <i class="fas fa-fw fa-map-marker-alt" aria-hidden="true"></i> <span itemprop="name" class="p-locality">Ludhina, Punjab <img class="emoji" title=":india:" alt=":india:" src="https://github.githubassets.com/images/icons/emoji/unicode/1f1ee-1f1f3.png" height="20" width="20"></span>
</li>
<li> <a href="mailto:shamshad.rattan@gmail.com" rel="me" class="u-email"><meta itemprop="email" content="shamshad.rattan@gmail.com"> <i class="fas fa-fw fa-envelope" aria-hidden="true"></i><span class="label">Email</span> </a>
</li>
<li><a href="https://www.researchgate.net/profile/Mohd-Shamshad" itemprop="sameAs" rel="nofollow noopener noreferrer me"> <i class="ai ai-researchgate-square" aria-hidden="true"></i><span class="label">ResearchGate</span> </a></li>
<li> <a href="https://twitter.com/DrShamshadM" itemprop="sameAs" rel="nofollow noopener noreferrer me"> <i class="fab fa-fw fa-twitter-square" aria-hidden="true"></i><span class="label">Twitter</span> </a>
</li>
<li> <a href="https://www.linkedin.com/in/dr-m-shamshad-58b4321b1" itemprop="sameAs" rel="nofollow noopener noreferrer me"> <i class="fab fa-fw fa-linkedin" aria-hidden="true"></i><span class="label">LinkedIn</span> </a>
</li>
<li><a href="https://scholar.google.com/citations?user=dI7V1a4AAAAJ&amp;hl=en" itemprop="sameAs" rel="nofollow noopener noreferrer me"> <i class="ai ai-fw ai-google-scholar-square" aria-hidden="true"></i><span class="label">Google Scholar</span></a></li>
</ul>
</div>
</div></div>
<article class="page" itemscope itemtype="https://schema.org/CreativeWork"><meta itemprop="headline" content="RNA-Seq-Based Transcriptomics Study to Investigate the Genes Governing Nitrogen Use Efficiency in Indian Wheat Cultivars">
<meta itemprop="description" content="High NUE (nitrogen use efficiency) has great practical significance for sustainable crop production. Wheat is one of the main cultivated crops worldwide for human food and nutrition. However, wheat grain productivity is dependent upon cultivars with high NUE in addition to the application of nitrogen fertilizers. In order to understand the molecular mechanisms exhibiting a high NUE response, a comparative transcriptomics study was carried out through RNA-seq analysis to investigate the gene expression that regulates NUE, in root and shoot tissue of N-efficient (PBW677) and N-inefficient (703) cultivars under optimum and nitrogen (N) stress. Differentially expressed gene analysis revealed a total of 2,406 differentially expressed genes (DEGs) present in both the contrasting cultivars under N stress. The efficient genotype PBW677 had considerably more abundant DEGs with 1,653 (903 roots +750 shoots) compared to inefficient cultivar PBW703 with 753 (96 roots +657 shoots). Gene ontology enrichment and pathway analysis of these DEGs suggested that the two cultivars differed in terms of adaptive mechanism. Gene enrichment analysis revealed that among the upregulated and downregulated genes the overrepresented and underrepresented gene categories belonged to biological processes like DNA binding, response to abiotic stimulus, photosynthesis, carbon fixation, carbohydrate metabolic process, nitrogen compound metabolic process, nitrate transport, and translation in cultivar PBW677, while the enriched biological processes were nucleosome assembly, chromatin remodeling, DNA packaging, lipid transport, sulfur compound metabolic process, protein modifications, and protein folding and refolding in N inefficient cultivar PBW703. We found several transcription factors (MYB, WRKY, RING finger protein, zinc finger protein, transporters, NRT1, amino acid transporters, sugar), protein kinases, and genes involved in N absorption, transportation, and assimilation to be highly expressed in high NUE cultivar PBW677. In our study, we report 13 potential candidate genes which showed alternate gene expression in the two contrasting cultivars under study. These genes could serve as potential targets for future breeding programs.">
<meta itemprop="datePublished" content="2022-03-31T00:00:00+05:30">
<meta itemprop="dateModified" content="2024-10-06T20:53:46+05:30">
<div class="page__inner-wrap">
<header><h1 id="page-title" class="page__title" itemprop="headline"> <a href="https://shamshad.in/publication/2022-03-31-RNA-Seq-NUE,-differential-gene-expression" itemprop="url">RNA-Seq-Based Transcriptomics Study to Investigate the Genes Governing Nitrogen Use Efficiency in Indian Wheat Cultivars </a>
</h1>
<p>Published in <i>Frontier in Genetics</i>, 2022</p></header><section class="page__content" itemprop="text"><p><a href="/files/research/RNA-seq.pdf">Download PDF here</a></p>
<p>Abstract: High NUE (nitrogen use efficiency) has great practical significance for sustainable crop production. Wheat is one of the main cultivated crops worldwide for human food and nutrition. However, wheat grain productivity is dependent upon cultivars with high NUE in addition to the application of nitrogen fertilizers. In order to understand the molecular mechanisms exhibiting a high NUE response, a comparative transcriptomics study was carried out through RNA-seq analysis to investigate the gene expression that regulates NUE, in root and shoot tissue of N-efficient (PBW677) and N-inefficient (703) cultivars under optimum and nitrogen (N) stress. Differentially expressed gene analysis revealed a total of 2,406 differentially expressed genes (DEGs) present in both the contrasting cultivars under N stress. The efficient genotype PBW677 had considerably more abundant DEGs with 1,653 (903 roots +750 shoots) compared to inefficient cultivar PBW703 with 753 (96 roots +657 shoots). Gene ontology enrichment and pathway analysis of these DEGs suggested that the two cultivars differed in terms of adaptive mechanism. Gene enrichment analysis revealed that among the upregulated and downregulated genes the overrepresented and underrepresented gene categories belonged to biological processes like DNA binding, response to abiotic stimulus, photosynthesis, carbon fixation, carbohydrate metabolic process, nitrogen compound metabolic process, nitrate transport, and translation in cultivar PBW677, while the enriched biological processes were nucleosome assembly, chromatin remodeling, DNA packaging, lipid transport, sulfur compound metabolic process, protein modifications, and protein folding and refolding in N inefficient cultivar PBW703. We found several transcription factors (MYB, WRKY, RING finger protein, zinc finger protein, transporters, NRT1, amino acid transporters, sugar), protein kinases, and genes involved in N absorption, transportation, and assimilation to be highly expressed in high NUE cultivar PBW677. In our study, we report 13 potential candidate genes which showed alternate gene expression in the two contrasting cultivars under study. These genes could serve as potential targets for future breeding programs.</p>
<p>Recommended citation: Kaur, S., <b>Shamshad, M.</b><sup id="fnref:1" role="doc-noteref"><a href="#fn:1" class="footnote" rel="footnote">1</a></sup>, Jindal, S., Kaur, A., Singh, S., sharma, A., &amp; Kaur, S. (2022). “RNA-Seq-Based Transcriptomics Study to Investigate the Genes Governing Nitrogen Use Efficiency in Indian Wheat Cultivars”. <i>Frontiers in Genetics</i>, 13(March), 1-15. Retrieved from https://doi.org/10.3389/fgene.2022.8539</p>
<div class="footnotes" role="doc-endnotes"><ol><li id="fn:1" role="doc-endnote"><p>First and Second author Equal authorship <a href="#fnref:1" class="reversefootnote" role="doc-backlink">↩</a></p></li></ol></div>
<p style="font-size: smaller">Recommended citation: Kaur, S., <b>Shamshad, M.</b><sup>*</sup>, Jindal, S., Kaur, A., Singh, S., sharma, A., &amp; Kaur, S. (2022). "RNA-Seq-Based Transcriptomics Study to Investigate the Genes Governing Nitrogen Use Efficiency in Indian Wheat Cultivars". <i>Frontiers in Genetics</i>, 13(March), 1-15. Retrieved from https://doi.org/10.3389/fgene.2022.8539</p></section><footer class="page__meta"><p class="page__date"><strong><i class="fas fa-fw fa-calendar-alt" aria-hidden="true"></i> Updated:</strong> <time datetime="2024-10-06">October 6, 2024</time></p></footer><section class="page__share"><h4 class="page__share-title">Share on</h4>
<a href="https://twitter.com/intent/tweet?via=DrShamshadM&amp;text=RNA-Seq-Based+Transcriptomics+Study+to+Investigate+the+Genes+Governing+Nitrogen+Use+Efficiency+in+Indian+Wheat+Cultivars%20https%3A%2F%2Fshamshad.in%2Fpublication%2F2022-03-31-RNA-Seq-NUE%2C-differential-gene-expression" class="btn btn--twitter" onclick="window.open(this.href, 'window', 'left=20,top=20,width=500,height=500,toolbar=1,resizable=0'); return false;" title="Share on Twitter"><i class="fab fa-fw fa-twitter" aria-hidden="true"></i><span> Twitter</span></a> <a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fshamshad.in%2Fpublication%2F2022-03-31-RNA-Seq-NUE%2C-differential-gene-expression" class="btn btn--facebook" onclick="window.open(this.href, 'window', 'left=20,top=20,width=500,height=500,toolbar=1,resizable=0'); return false;" title="Share on Facebook"><i class="fab fa-fw fa-facebook" aria-hidden="true"></i><span> Facebook</span></a> <a href="https://www.linkedin.com/shareArticle?mini=true&amp;url=https://shamshad.in/publication/2022-03-31-RNA-Seq-NUE,-differential-gene-expression" class="btn btn--linkedin" onclick="window.open(this.href, 'window', 'left=20,top=20,width=500,height=500,toolbar=1,resizable=0'); return false;" title="Share on LinkedIn"><i class="fab fa-fw fa-linkedin" aria-hidden="true"></i><span> LinkedIn</span></a></section><nav class="pagination"> <a href="https://shamshad.in/publication/2022-03-27-Tackling-a-Cereal-Killer-on-the-Run" class="pagination--pager" title="Tackling a Cereal Killer on the Run: Unending Fight Between Wheat Breeding and Foliar Rusts ">Previous</a> <a href="https://shamshad.in/publication/2022-04-06-Gpc-Gy-Pyramiding-genes" class="pagination--pager" title="Pyramiding of genes for grain protein content, grain quality, and rust resistance in eleven Indian bread wheat cultivars: a multi-institutional effort ">Next</a></nav>
</div></article>
</div></div><div id="footer" class="page__footer"><footer> <!-- start custom footer snippets --> <!-- end custom head snippets --> <!-- end custom footer snippets --><div class="page__footer-follow"><ul class="social-icons">
<li><strong>Follow:</strong></li>
<li><a href="https://twitter.com/DrShamshadM" rel="nofollow noopener noreferrer"><i class="fab fa-fw fa-twitter-square" aria-hidden="true"></i> Twitter</a></li>
<li><a href="https://scholar.google.com/citations?user=dI7V1a4AAAAJ&amp;hl=en" rel="nofollow noopener noreferrer"><i class="ai ai-google-scholar-square ai-fw" aria-hidden="true"></i> Google Scholar</a></li>
<li><a href="https://www.researchgate.net/profile/Mohd-Shamshad" rel="nofollow noopener noreferrer"><i class="ai ai-researchgate-square ai-fw" aria-hidden="true"></i> ResearchGate</a></li>
<li><a href="/feed.xml"><i class="fas fa-fw fa-rss-square" aria-hidden="true"></i> Feed</a></li>
</ul></div>
<div class="page__footer-copyright">© 2025 M. Shamshad. Proudly hosted by <a href="https://jekyllrb.com" rel="nofollow">Jekyll</a> &amp; <a href="https://mademistakes.com/work/minimal-mistakes-jekyll-theme/" rel="nofollow">Minimal Mistakes</a>.</div>
<link defer rel="preload" href="/assets/js/mermaid.min.js" as="script"> <script defer src="/assets/js/mermaid.min.js"></script></footer></div><link defer rel="preload" href="/assets/js/main.min.js" as="script"> <script defer src="/assets/js/main.min.js"></script> <!-- <script src="https://kit.fontawesome.com/4eee35f757.js"></script> --> <script defer src="/assets/js/mermaid.js"></script></body></html>
