<!doctype html><html lang="en-US" class="no-js"><head><meta charset="utf-8"> <!-- begin _includes/seo.html --><title>Publications | M. Shamshad</title><meta name="description" content="I have dream…"><meta name="author" content="M. Shamshad"><meta property="og:type" content="website"><meta property="og:locale" content="en_US"><meta property="og:site_name" content="M. Shamshad"><meta property="og:title" content="Publications"><meta property="og:url" content="https://shamshad.in/publications/"><meta property="og:description" content="I have dream…"><meta property="og:image" content="https://shamshad.in/images/favicon/shamshad.webp"><meta name="twitter:site" content="@DrShamshadM"><meta name="twitter:title" content="Publications"><meta name="twitter:description" content="I have dream…"><meta name="twitter:url" content="https://shamshad.in/publications/"><meta name="twitter:card" content="summary_large_image"><meta name="twitter:image" content="https://shamshad.in/images/favicon/shamshad.webp"><meta name="twitter:creator" content="@DrShamshadM"><link rel="canonical" href="https://shamshad.in/publications/"> <!-- end _includes/seo.html --><link href="/feed.xml" type="application/atom+xml" rel="alternate" title="M. Shamshad Feed"> <!-- https://web.dev/viewport/?utm_source=lighthouse&utm_medium=lr --><meta name="viewport" content="width=device-width, initial-scale=1.0"> <script defer> document.documentElement.className = document.documentElement.className.replace(/\bno-js\b/g, '') + ' js '; window.enable_copy_code_button = true; </script><link rel="stylesheet" href="/assets/academicons-1.9.4/css/academicons.min.css"/> <!-- For all browsers --><link rel="stylesheet" href="/assets/css/main.css" id="theme_source" /><link rel="stylesheet alternate" href="/assets/css/mainDark.css" id="theme_source_2" /> <script async> let theme = sessionStorage.getItem("theme"); if (theme === "dark") { sessionStorage.setItem("theme", "dark"); node1 = document.getElementById("theme_source"); node2 = document.getElementById("theme_source_2"); node1.setAttribute("rel", "stylesheet alternate"); node2.setAttribute("rel", "stylesheet"); } else { sessionStorage.setItem("theme", "light"); } </script> <!-- start custom head snippets --><meta name="apple-mobile-web-app-capable" content="yes" /><meta name="apple-mobile-web-app-status-bar-style" content="default" /><link rel="apple-touch-icon" sizes="60x60" href="/images/favicon/apple-touch-icon-60x60.png?v=565802"><link rel="apple-touch-icon" sizes="76x76" href="/images/favicon/apple-touch-icon-76x76.png?v=565802"><link rel="apple-touch-icon" sizes="120x120" href="/images/favicon/apple-touch-icon-120x120.png?v=565802"><link rel="apple-touch-icon" sizes="152x152" href="/images/favicon/apple-touch-icon-152x152.png?v=565802"><link rel="apple-touch-icon" sizes="180x180" href="/images/favicon/apple-touch-icon-180x180.png?v=565802"><link rel="icon" type="image/png" sizes="32x32" href="/images/favicon/favicon-32x32.png?v=565802"><link rel="icon" type="image/png" sizes="16x16" href="/images/favicon/favicon-16x16.png?v=565802"><link rel="manifest" href="/images/favicon/site.webmanifest?v=565802"><link rel="mask-icon" href="/images/favicon/safari-pinned-tab.svg?v=565802" color="#00a7ff"><link rel="shortcut icon" href="/images/favicon/favicon.ico?v=565802"><meta name="apple-mobile-web-app-title" content="SM-Web"><meta name="application-name" content="SM-Web"><meta name="msapplication-TileColor" content="#00a300"><meta name="msapplication-config" content="/images/favicon/browserconfig.xml?v=565802"><meta name="theme-color" content="#ffffff"> <!-- SEO tags --> <!-- Begin Jekyll SEO tag v2.8.0 --><title>Publications | M. Shamshad</title><meta name="generator" content="Jekyll v4.3.4" /><meta property="og:title" content="Publications" /><meta name="author" content="M. Shamshad" /><meta property="og:locale" content="en_US" /><meta name="description" content="I have dream…" /><meta property="og:description" content="I have dream…" /><link rel="canonical" href="https://shamshad.in/publications/" /><meta property="og:url" content="https://shamshad.in/publications/" /><meta property="og:site_name" content="M. Shamshad" /><meta property="og:type" content="website" /><meta name="twitter:card" content="summary" /><meta property="twitter:title" content="Publications" /><meta name="twitter:site" content="@DrShamshadM" /><meta name="twitter:creator" content="@DrShamshadM" /> <script type="application/ld+json"> {"@context":"https://schema.org","@type":"WebPage","author":{"@type":"Person","name":"M. Shamshad"},"dateModified":"2024-10-06T20:53:46+05:30","description":"I have dream…","headline":"Publications","publisher":{"@type":"Organization","logo":{"@type":"ImageObject","url":"https://shamshad.in/images/favicon/site-treaser.webp"},"name":"M. Shamshad"},"url":"https://shamshad.in/publications/"}</script> <!-- End Jekyll SEO tag --></head><body class="layout--single wide" dir="ltr"><nav class="skip-links"><ul>
<li><a href="#site-nav" class="screen-reader-shortcut">Skip to primary navigation</a></li>
<li><a href="#main" class="screen-reader-shortcut">Skip to content</a></li>
<li><a href="#footer" class="screen-reader-shortcut">Skip to footer</a></li>
</ul></nav><div class="masthead"><div class="masthead__inner-wrap"><div class="masthead__menu"><nav id="site-nav" class="greedy-nav"> <a class="site-logo" href="/"><img src="/images/favicon/site-treaser.webp" alt="M. Shamshad"></a> <a class="site-title" href="/"> M. Shamshad </a><ul class="visible-links">
<li class="masthead__menu-item"> <a href="/about/">About</a>
</li>
<li class="masthead__menu-item"> <a href="/teaching/">Teaching</a>
</li>
<li class="masthead__menu-item"> <a href="/publications/">Publications</a>
</li>
<li class="masthead__menu-item"> <a href="/talks/">Talks</a>
</li>
<li class="masthead__menu-item"> <a href="/research/">Research</a>
</li>
<li class="masthead__menu-item"> <a href="/resume/">Resume</a>
</li>
<li class="masthead__menu-item"> <a href="/tags/">Tags</a>
</li>
<li class="masthead__menu-item"> <a href="/docs/Linux_tutorial/">Bioinfo</a>
</li>
</ul>
<span class="fa-solid fa-moon toggle-switch" type="button" style="cursor: pointer;" aria-hidden="true" onclick="node1 = document.getElementById('theme_source'); node2 = document.getElementById('theme_source_2'); if(node1.getAttribute('rel')=='stylesheet'){node1.setAttribute('rel', 'stylesheet alternate'); node2.setAttribute('rel', 'stylesheet'); sessionStorage.setItem('theme', 'dark');} else{node2.setAttribute('rel', 'stylesheet alternate'); node1.setAttribute('rel', 'stylesheet');sessionStorage.setItem('theme', 'light');} return false;"></span> <button class="greedy-nav__toggle hidden" type="button"> <span class="visually-hidden">Toggle menu</span><div class="navicon"></div></button><ul class="hidden-links hidden"></ul></nav></div></div></div><div class="initial-content">
<div loading="lazy" class="page__hero--overlay" style=" background-image: url('https://shamshad.in/images//header/publications.jpeg');"><div class="wrapper"><h1 id="page-title" class="page__title" itemprop="headline"> Publications</h1></div></div>
<div id="main" role="main">
<div class="sidebar sticky"><div itemscope itemtype="https://schema.org/Person" class="h-card">
<div class="author__avatar"> <a href="https://shamshad.in/"> <img loading="lazy" src="/images/profile.jpeg" alt="M. Shamshad" itemprop="image" class="u-photo"> </a>
</div>
<div class="author__content">
<h3 class="author__name p-name" itemprop="name"> <a class="u-url" rel="me" href="https://shamshad.in/" itemprop="url">M. Shamshad</a>
</h3>
<div class="author__bio p-note" itemprop="description"><p>Plant Breeding &amp;&amp; Data Science</p></div>
</div>
<div class="author__urls-wrapper"> <button class="btn btn--inverse">Follow</button><ul class="author__urls social-icons">
<li itemprop="homeLocation" itemscope itemtype="https://schema.org/Place"> <i class="fas fa-fw fa-map-marker-alt" aria-hidden="true"></i> <span itemprop="name" class="p-locality">Ludhina, Punjab, India <img class="emoji" title=":india:" alt=":india:" src="https://github.githubassets.com/images/icons/emoji/unicode/1f1ee-1f1f3.png" height="20" width="20"></span>
</li>
<li> <a href="mailto:shamshad.rattan@gmail.com" rel="me" class="u-email"><meta itemprop="email" content="shamshad.rattan@gmail.com"> <i class="fas fa-fw fa-envelope-square" aria-hidden="true"></i><span class="label">Email</span> </a>
</li>
<li><a href="https://www.researchgate.net/profile/Mohd-Shamshad" itemprop="sameAs" rel="nofollow noopener noreferrer me"> <i class="ai ai-researchgate-square" aria-hidden="true"></i><span class="label"> ResearchGate</span> </a></li>
<li> <a href="https://twitter.com/DrShamshadM" itemprop="sameAs" rel="nofollow noopener noreferrer me"> <i class="fab fa-fw fa-twitter-square" aria-hidden="true"></i><span class="label">Twitter</span> </a>
</li>
<li> <a href="https://www.linkedin.com/in/dr-m-shamshad-58b4321b1" itemprop="sameAs" rel="nofollow noopener noreferrer me"> <i class="fab fa-fw fa-linkedin" aria-hidden="true"></i><span class="label">LinkedIn</span> </a>
</li>
<li><a href="https://scholar.google.com/citations?user=dI7V1a4AAAAJ&amp;hl=en" itemprop="sameAs" rel="nofollow noopener noreferrer me"> <i class="ai ai-fw ai-google-scholar-square" aria-hidden="true"></i><span class="label">Google Scholar</span></a></li>
</ul>
</div>
</div></div>
<article class="page" itemscope itemtype="https://schema.org/CreativeWork"><meta itemprop="headline" content="Publications">
<meta itemprop="dateModified" content="2024-10-06T20:53:46+05:30">
<div class="page__inner-wrap">
<section class="page__content" itemprop="text"><hr>
<p><i class="ai ai-ideas-repec ai-2x"> Pre-prints</i></p>
<hr>
<p>Comming soon</p>
<hr>
<p><a href="https://scholar.google.com/citations?user=dI7V1a4AAAAJ&amp;hl=en"><i class="ai ai-google-scholar ai-2x"></i></a></p>
<p>I have collaborated to manuscripts published in high quality peer-reviewed scientific journals.</p>
<hr>
<h2>Peer-Reviewed Publications</h2>
<div class="list__item"><article class="archive__item" itemscope="" itemtype="https://schema.org/CreativeWork"><h2 class="archive__item-title" itemprop="headline"> <a href="https://shamshad.in/publication/2022-12-16-%C2%A0PULSE_project-Phyt" rel="permalink">Variability in iron, zinc, phytic acid and protein content in pre‑breeding wheat germplasm under diferent water regimes </a>
</h2>
<p>Published in <i>Plant Growth Regulation</i>, 2022</p>
<p class="archive__item-excerpt" itemprop="description"></p>
<p>More than one-third of the global population sufers from iron and zinc defciency, developing anaemia like diseases mostly in the developing countries. Therefore, the current study focuses on the investigation of variability and bioavailability of micronutrients such as iron and zinc in bread wheat mediated by lower levels of phytic acid which is often categorised as an antinutritional compound. Phytic acid in cereals acts as an chelator of major micronutrients such as iron and zinc, thus lowering their bioavailability both in humans and animals. In addition, drought is a major component afecting the micronutrient accumulation in wheat kernels. Therefore, a pre-breeding wheat germplasm set comprising 137 genotypes was grown under irrigated and restricted irrigated conditions for 2 years. This germplasm set was used to assess the variability for iron, zinc and phytic acid content in the wheat kernels. Mean iron and zinc content was 45.83 and 49.43 ppm under irrigated conditions, whereas it was 40.53 and 49.62 ppm under restricted irrigated conditions. Afterward, the molar ratios of phytate with iron and zinc were calculated to predict their bioavailability. Based on the daily recommended values, promising genotypes were shortlisted with low phytic acid combined with high iron and zinc content. These promising genotypes will be further used in wheat breeding programme to breed biofortifed wheat cultivars with higher micronutrient and reduced phytic acid concentration combined with enchanted abiotic stress tolerance which can potentially help in alleviating the hidden hunger under changing climatic conditions.</p>
<p>Recommended citation: Sharma, H., Singh, S., <b>Shamshad, M. </b> et al. (2022) Variability in iron, zinc, phytic acid and protein content in pre-breeding wheat germplasm under different water regimes.<i>Plant Growth Regul </i>. https://doi.org/10.1007/s10725-022-00943-5</p>
<!-- citation and icon code --> <a href="https://doi.org/10.1007/s10725-022-00943-5"><i class="fas fa-fw fa-link zoom" aria-hidden="true"></i></a> <a href="/files/research/s10725-022-00943-5.pdf"><i class="fas fa-fw fa-file-pdf zoom" aria-hidden="true"></i></a></article></div>
<div class="list__item"><article class="archive__item" itemscope="" itemtype="https://schema.org/CreativeWork"><h2 class="archive__item-title" itemprop="headline"> <a href="https://shamshad.in/publication/2022-07-31-LTP-NUE-GWAS" rel="permalink">Preliminary Dissection of Grain Yield and Related Traits at Diferential Nitrogen Levels in Diverse Pre-Breeding Wheat Germplasm Through Association Mapping </a>
</h2>
<p>Published in <i>Molecular Biotechnology</i>, 2022</p>
<p class="archive__item-excerpt" itemprop="description"></p>
<p>Development of nutrient efficient cultivars depends on effective identification and utilization of genetic variation. We characterized a set of 276 pre-breeding lines (PBLs) for several traits at different levels of nitrogen application. These PBLs originate from synthetic wheats and landraces. We witnessed significant variation in various traits among PBLs to different nitrogen doses. There was ~ 4–18% variation range in different agronomic traits in response to nitrogen application, with the highest variation for the biological yield (BY) and the harvest index. Among various agronomic traits measured, plant height, tiller number, and BY showed a positive correlation with nitrogen applications. GWAS analysis detected 182 marker-trait associations (MTAs) (at p-value &lt; 0.001), out of which 8 MTAs on chromosomes 5D, 4A, 6A, 1B, and 5B explained more than 10% phenotypic variance. Out of all, 40 MTAs observed for differential nitrogen application response were contributed by the synthetic derivatives. Moreover, 20 PBLs exhibited significantly higher grain yield than checks and can be selected as potential donors for improved plant nitrogen use efficiency (pNUE).</p>
<p>Recommended citation: Sharma, A., Arif, M. A. R., <b>Shamshad, M.</b>, Rawale, K. S., Brar, A., Burgueno, J., … Singh, S. (2022). "Preliminary Dissection of Grain Yield and Related Traits at Differential Nitrogen Levels in Diverse Pre-Breeding Wheat Germplasm Through Association Mapping".<i> Molecular Biotechnology</i>. Retrieved from https://doi.org/10.1007/s12033-022-00535-8</p>
<!-- citation and icon code --> <a href="https://doi.org/10.1007/s12033-022-00535-8"><i class="fas fa-fw fa-link zoom" aria-hidden="true"></i></a> <a href="/files/research/LTP_N_association_Mapping.pdf"><i class="fas fa-fw fa-file-pdf zoom" aria-hidden="true"></i></a></article></div>
<div class="list__item"><article class="archive__item" itemscope="" itemtype="https://schema.org/CreativeWork"><h2 class="archive__item-title" itemprop="headline"> <a href="https://shamshad.in/publication/2022-05-17-Carotenoid-biofortification" rel="permalink">In silico characterization of <i>Thinopyrum elongatum</i> derived PsyE1 gene and validation in 7D/7E bread wheat introgression lines open avenues for carotenoid biofortifcation in wheat </a>
</h2>
<p>Published in <i>Journal of Cereal Research Communications</i>, 2022</p>
<p class="archive__item-excerpt" itemprop="description"></p>
<p>Current global scenario demands agricultural productivity of food grains to be kept at abreast with burgeoning population. Cereals constitute major food stuff for millennia and biofortification of new cereal varieties provides an opportunity to tackle global-scale malnutrition deficiencies without doing major shifts in the diets. Carotenoid biofortification in wheat grains has recently caught the attention of breeders owing to a myriad of health benefits offered by this micronutrient. Thinopyrum elongatum-derived PsyE1 gene encoding for Phytoene Synthase encoding Y gene, is a jackpot to enhance the carotenoid content in wheat. The present study is the first report deciphering detailed in silico characterization of Thinopyrum elongatum-derived PsyE1 gene and its protein. Promoter analysis of chloroplast localized PsyE1 gene provides clues about its possible role in stress resistance along with enhancing the carotenoid content in both durum and bread wheat. Homology, phylogeny and protein modelling studies of PsyE1 revealed its closer evolutionary relationship with barley and wheat, as well as provided a preliminary insight into catalytic and secondary structure of the protein. PCR validation of PsyE1 in 7D/7E bread wheat introgression lines further facilitated development of functional marker that could be used to track its introgression in elite bread wheat varieties. Overall, these detailed insilico insights into structure, function and validation of PsyE1 open doors for its deployment in to produce carotene biofortified hexaploid wheat through facilitating development of functional markers and MAS, as well as to elucidate its mechanism of action and regulation in response to external stimuli.</p>
<p>Recommended citation: Padhy, A. K., Kaur, P., Singh, B., Kaur, R., Bhatia, S., <b>Shamshad, M.</b>, Sharma, A. (2022) "In silico characterization of Thinopyrum elongatum-derived PsyE1 gene and validation in 7D/7E bread wheat introgression lines open avenues for carotenoid biofortification in wheat". <i>Cereal Research Communications</i>, (0123456789). Retrieved from https://doi.org/10.1007/s42976-022-00279-w</p>
<!-- citation and icon code --> <a href="https://doi.org/10.1007/s42976-022-00279-w"><i class="fas fa-fw fa-link zoom" aria-hidden="true"></i></a> <a href="/files/research/Y_Gene_Transfer.pdf"><i class="fas fa-fw fa-file-pdf zoom" aria-hidden="true"></i></a></article></div>
<div class="list__item"><article class="archive__item" itemscope="" itemtype="https://schema.org/CreativeWork"><h2 class="archive__item-title" itemprop="headline"> <a href="https://shamshad.in/publication/2022-05-05-Meta-Analysis-quality-trait" rel="permalink">Unravelling consensus genomic regions associated with quality traits in wheat using meta-analysis of quantitative trait loci </a>
</h2>
<p>Published in <i>Planta</i>, 2022</p>
<p class="archive__item-excerpt" itemprop="description"></p>
<p>A meta-analysis of quantitative trait loci (QTL) associated with dough rheology properties, nutritional traits, and processing quality traits was conducted in wheat. For this purpose, as many as 2458 QTL were collected from 50 interval mapping studies published during 2013–2020. Of the total QTL, 1126 QTL were projected onto the consensus map saturated with 249,603 markers which led to the identification of 110 meta-QTL (MQTL). These MQTL exhibited an 18.84-fold reduction in the average CI compared to the average CI of the initial QTL (ranging from 14.87 to 95.55 cM with an average of 40.35 cM). Of the 110, 108 MQTL were physically anchored to the wheat reference genome, including 51 MQTL verified with marker-trait associations (MTAs) reported from earlier genome-wide association studies. Candidate gene (CG) mining allowed the identification of 2533 unique gene models from the MQTL regions. In-silico expression analysis discovered 439 differentially expressed gene models with &gt; 2 transcripts per million expressions in grains and related tissues, which also included 44 high-confidence CGs involved in the various cellular and biochemical processes related to quality traits. Nine functionally characterized wheat genes associated with grain protein content, high-molecular-weight glutenin, and starch synthase enzymes were also found to be co-localized with some of the MQTL. Synteny analysis between wheat and rice MQTL regions identified 23 wheat MQTL syntenic to 16 rice MQTL associated with quality traits. Furthermore, 64 wheat orthologues of 30 known rice genes were detected in 44 MQTL regions. Markers flanking the MQTL identified in the present study can be used for marker-assisted breeding and as fixed effects in the genomic selection models for improving the prediction accuracy during quality breeding. Wheat orthologues of rice genes and other CGs available from MQTLs can be promising targets for further functional validation and to better understand the molecular mechanism underlying the quality traits in wheat.</p>
<p>Recommended citation: Gudi, S., Saini, D. K., Singh, G., Halladakeri, P., Kumar, P., <b>Shamshad, M.</b>, Sharma, A. (2022) "Unravelling consensus genomic regions associated with quality traits in wheat using meta-analysis of quantitative trait loci". <i>Planta</i>, 255(6), 1-19. Retrieved from https://doi.org/10.1007/s00425-022-03904-4</p>
<!-- citation and icon code --> <a href="https://doi.org/10.1007/s00425-022-03904-4"><i class="fas fa-fw fa-link zoom" aria-hidden="true"></i></a> <a href="/files/research/Meta-Analysis-GPC.pdf"><i class="fas fa-fw fa-file-pdf zoom" aria-hidden="true"></i></a></article></div>
<div class="list__item"><article class="archive__item" itemscope="" itemtype="https://schema.org/CreativeWork"><h2 class="archive__item-title" itemprop="headline"> <a href="https://shamshad.in/publication/2022-04-06-Gpc-Gy-Pyramiding-genes" rel="permalink">Pyramiding of genes for grain protein content, grain quality, and rust resistance in eleven Indian bread wheat cultivars: a multi-institutional effort </a>
</h2>
<p>Published in <i>Molecular Breeding</i>, 2022</p>
<p class="archive__item-excerpt" itemprop="description"></p>
<p>Improvement of grain protein content (GPC), loaf volume, and resistance to rusts was achieved in 11 Indian wheat cultivars that are widely grown in four different agro-climatic zones of India. This involved use of marker-assisted backcross breeding (MABB) for introgression and pyramiding of the following genes: (i) the high GPC gene Gpc-B1; (ii) HMW glutenin subunits 5 + 10 at Glu-D1 loci, and (iii) rust resistance genes, Yr36, Yr15, Lr24, and Sr24. GPC increased by 0.8 to 3.3%, although high GPC was generally associated with yield penalty. Further selection among high GPC lines allowed identification of progenies with higher GPC associated with improvement in 1000-grain weight and grain yield in the backgrounds of the following four cultivars: NI5439, UP2338, UP2382, and HUW468. The high GPC progenies (derived from NI5439) were also improved for grain quality using HMW glutenin subunits 5 + 10 at Glu-D1 loci. Similarly, progenies combining high GPC and rust resistance were obtained in the backgrounds of following five cultivars: Lok1, HD2967, PBW550, PBW621, and DBW1. The improved pre-bred lines developed following multi-institutional effort should prove a valuable source for the development of cultivars with improved nutritional quality and rust resistance in the ongoing wheat breeding programmes.</p>
<p>Recommended citation: Gupta, P. K., Balyan, H. S., Chhuneja, P., Jaiswal, J. P., Tamhankar, S., Mishra, V. K., Vishwakarma, M. K. (2022) "Pyramiding of genes for grain protein content, grain quality, and rust resistance in eleven Indian bread wheat cultivars: a multi-institutional effort". <i>Molecular Breeding</i>, 42(4), 116. Retrieved from https://doi.org/10.1007/s11032-022-01277</p>
<!-- citation and icon code --> <a href="https://doi.org/10.1007/s11032-022-01277-w"><i class="fas fa-fw fa-link zoom" aria-hidden="true"></i></a> <a href="/files/research/Pyramiding-Gpc-Yield.pdf"><i class="fas fa-fw fa-file-pdf zoom" aria-hidden="true"></i></a></article></div>
<div class="list__item"><article class="archive__item" itemscope="" itemtype="https://schema.org/CreativeWork"><h2 class="archive__item-title" itemprop="headline"> <a href="https://shamshad.in/publication/2022-03-31-RNA-Seq-NUE,-differential-gene-expression" rel="permalink">RNA-Seq-Based Transcriptomics Study to Investigate the Genes Governing Nitrogen Use Efficiency in Indian Wheat Cultivars </a>
</h2>
<p>Published in <i>Frontier in Genetics</i>, 2022</p>
<p class="archive__item-excerpt" itemprop="description"></p>
<p>High NUE (nitrogen use efficiency) has great practical significance for sustainable crop production. Wheat is one of the main cultivated crops worldwide for human food and nutrition. However, wheat grain productivity is dependent upon cultivars with high NUE in addition to the application of nitrogen fertilizers. In order to understand the molecular mechanisms exhibiting a high NUE response, a comparative transcriptomics study was carried out through RNA-seq analysis to investigate the gene expression that regulates NUE, in root and shoot tissue of N-efficient (PBW677) and N-inefficient (703) cultivars under optimum and nitrogen (N) stress. Differentially expressed gene analysis revealed a total of 2,406 differentially expressed genes (DEGs) present in both the contrasting cultivars under N stress. The efficient genotype PBW677 had considerably more abundant DEGs with 1,653 (903 roots +750 shoots) compared to inefficient cultivar PBW703 with 753 (96 roots +657 shoots). Gene ontology enrichment and pathway analysis of these DEGs suggested that the two cultivars differed in terms of adaptive mechanism. Gene enrichment analysis revealed that among the upregulated and downregulated genes the overrepresented and underrepresented gene categories belonged to biological processes like DNA binding, response to abiotic stimulus, photosynthesis, carbon fixation, carbohydrate metabolic process, nitrogen compound metabolic process, nitrate transport, and translation in cultivar PBW677, while the enriched biological processes were nucleosome assembly, chromatin remodeling, DNA packaging, lipid transport, sulfur compound metabolic process, protein modifications, and protein folding and refolding in N inefficient cultivar PBW703. We found several transcription factors (MYB, WRKY, RING finger protein, zinc finger protein, transporters, NRT1, amino acid transporters, sugar), protein kinases, and genes involved in N absorption, transportation, and assimilation to be highly expressed in high NUE cultivar PBW677. In our study, we report 13 potential candidate genes which showed alternate gene expression in the two contrasting cultivars under study. These genes could serve as potential targets for future breeding programs.</p>
<p>Recommended citation: Kaur, S., <b>Shamshad, M.</b><sup>*</sup>, Jindal, S., Kaur, A., Singh, S., sharma, A., &amp; Kaur, S. (2022). "RNA-Seq-Based Transcriptomics Study to Investigate the Genes Governing Nitrogen Use Efficiency in Indian Wheat Cultivars". <i>Frontiers in Genetics</i>, 13(March), 1-15. Retrieved from https://doi.org/10.3389/fgene.2022.8539</p>
<!-- citation and icon code --> <a href="https://doi.org/10.3389/fgene.2022.853910"><i class="fas fa-fw fa-link zoom" aria-hidden="true"></i></a> <a href="/files/research/RNA-seq.pdf"><i class="fas fa-fw fa-file-pdf zoom" aria-hidden="true"></i></a></article></div>
<div class="list__item"><article class="archive__item" itemscope="" itemtype="https://schema.org/CreativeWork"><h2 class="archive__item-title" itemprop="headline"> <a href="https://shamshad.in/publication/2022-02-19-PAU-Salinity" rel="permalink">Salt stress genotypic response: Relative tolerance of wheat cultivars to salinity </a>
</h2>
<p>Published in <i>Agricultural Research Journal</i>, 2022</p>
<p class="archive__item-excerpt" itemprop="description"></p>
<p>The commercial wheat genotypes released for cultivation under timely sown irrigated conditions in Punjab along with known salt tolerant genotypes and few advance breeding lines were evaluated for salt stress tolerance at seedling stage under simulated stress conditions and at adult plant stage in actual saline fields at Rattakhera, Sri Mukatsar Sahib (Punjab), to find out the best suitable wheat variety for the salt affected areas. Twelve wheat varieties, seven advance breeding lines, one wheat genotype popular in salt stressed regions of Punjab and Kharchia, a universal donor for stress tolerance to salt were tested. PBW 765 and PBW 780 had highest tolerance among advance germplasm, HD 3086, HD 2967, KRL 210 could be categorized in one group whereas PBW 725 showed moderate tolerance to salt stress. These identified genotypes are most suitable for regions having moderate to high salinity stress as these were less affected due to salinity.</p>
<p>Recommended citation: Sharma, A., Ankita, <b>Shamshad, M.</b>, Kaur, S., Singh, A., Srivastava, P., Sohu, V. S. (2021) "SALT STRESS GENOTYPIC RESPONSE: RELATIVE TOLERANCE OF WHEAT CULTIVARS TO SALINITY". <i>Agricultural Research Journal</i>, 58(6), 974-982. Retrieved from https://doi.org/10.5958/2395-146X.2021.00138</p>
<!-- citation and icon code --> <a href="https://doi.org/10.5958/2395-146X.2021.00138.1"><i class="fas fa-fw fa-link zoom" aria-hidden="true"></i></a> <a href="/files/research/ARJ-4018.pdf"><i class="fas fa-fw fa-file-pdf zoom" aria-hidden="true"></i></a></article></div>
<div class="list__item"><article class="archive__item" itemscope="" itemtype="https://schema.org/CreativeWork"><h2 class="archive__item-title" itemprop="headline"> <a href="https://shamshad.in/publication/2022-01-22-Salinity-lranian" rel="permalink">Uncovering the Iranian wheat landraces for salinity stress tolerance at early stages of plant growth </a>
</h2>
<p>Published in <i>Journal of Cereal Research Communications</i>, 2022</p>
<p class="archive__item-excerpt" itemprop="description"></p>
<p>Soil salinity is a major global environmental factor limiting plant growth and productivity. Wheat seeds need to be able to germinate and establish seedlings in saline soils for sustained productivity. In this study, we investigated seed germination-related traits under salt stress conditions in 239 diverse Iranian wheat landraces for evaluation of salt stress tolerance. Seed of the landraces along with relevant checks was germinated in salt and control solutions until 14 days. Initially, 10 randomly selected accessions were subjected to six different (25 mM, 50 mM, 75 mM, 100 mM, 125 mM, 150 mM) salinity levels for standardization. The salinity level 125 mM NaCl was found more effective concentration for the discrimination of genotypes for various physiological indices, viz. germination percentage, coleoptile length, root and shoot length, fresh root and shoot weight, dry root and shoot weight, and vigor index. After 14 days, germination percentage and all seedling traits were found to be affected due to salinity. Salt tolerance index maintained a significant positive correlation with seedling traits which indicates that these parameters could be used as selection criteria for screening wheat genotypes against salt stress. Significant differences were observed for coleoptile length, root–shoot length, fresh root–a shoot weight, dry shoot weight, and vigor index among the wheat landraces. From the overall observation of germination percentage and early seedling growth, it was concluded that the wheat landraces accessions including IWA 8600278, IWA 8600291, IWA 8611786, IWA 8600179, IWA 8600303, and IWA 8610487 showed better salt tolerance than Kharchia 65, the universal salt-tolerant variety used so far in wheat-breeding programs.</p>
<p>Recommended citation: Kaur, S., Suhalia, A., Sarlach, R. S., <b>Shamshad, M.</b>, Singh, P., Grover, G., Sharma, A. (2022)"Uncovering the Iranian wheat landraces for salinity stress tolerance at early stages of plant growth". <i>Cereal Research Communications </i>. Retrieved from https://doi.org/10.1007/s42976-022-00245-6</p>
<!-- citation and icon code --> <a href="https://doi.org/10.1007/s42976-022-00245-6"><i class="fas fa-fw fa-link zoom" aria-hidden="true"></i></a> <a href="/files/research/Irianian_Wheat_Salinity.pdf"><i class="fas fa-fw fa-file-pdf zoom" aria-hidden="true"></i></a></article></div>
<div class="list__item"><article class="archive__item" itemscope="" itemtype="https://schema.org/CreativeWork"><h2 class="archive__item-title" itemprop="headline"> <a href="https://shamshad.in/publication/2016-05-18-Heterosis-for-oil-and-oil-content" rel="permalink">Heterosis for oil content and oil quality in sunflower (<i>Helianthus annuus </i>L.) </a>
</h2>
<p>Published in <i>Current Advances in Agricultural Sciences</i>, 2016</p>
<p class="archive__item-excerpt" itemprop="description"></p>
<p>The present study was aimed to estimate heterosis for oil content and oil quality of the newly developed hybrids of sunflower (Helianthus annuus L.) using selected parental lines. For this purpose five cytoplasmic male sterile lines CMS 11A, CMS 47A, CMS 67A, CMS 68A and CMS 234A and eight restorer lines 95 C-1, P 93R, P 103R, P 124R, P 134R, P 145R, P 167R and RCR 8297 and their forty F1 hybrids along with commercial check PSH 569 were evaluated in randomized complete block design with three replications at Ludhiana during 2014. For oil content the magnitude of mid-parent heterosis ranged from -5.25% (CMS 11A x 95 C-1) to 21.40% (CMS 234A x P167R). The value for heterobeltiosis varied from -12.45% (CMS 11A x 95C-1) to 13.25% (CMS 11A x P 103R). The standard heterosis ranged from -22.77% (CMS 11A x P 145R) to 1.73% (CMS 67A x P 134R) over PSH 569. Mid, high and standard heterosis estimates for oil quality i.e. fatty acid composition of F1 hybrids ranged from -27.94% to -28.57%, -22.81% to -37.18%, -2% to -44% for palmitic acid (C16:0); from -47.73% to -18.03%, -50.98% to -2.27%, -44.74% to -13.16% for stearic acid (C18:0); from - 35.06% to -36.91%, -49.15% to -16.44%, -59.86% to -8.17% for oleic acid (C18:1) and from -26.12% to -55.03%, -35.93% to -34.28%, -4.52% to -115.81% for linoleic acid (C18:2), respectively. In the present investigation, fourteen out of the forty experimental hybrids recorded significantly positive heterosis over mid parent, whereas twenty hybrids recorded significant positive heterosis over better parent for oleic acid. The range of standard heterosis over check hybrids PSH 569 was from -59.86% to 8.17%. Among the hybrids, only one CMS234A x 95 C-1 had significantly positive heterosis (8.17%) over standard check hybrid PSH 569. Thus, the hybrids showed significant standard heterosis for both oil content and quality, therefore, these parental lines may further be used for improvement of oil content and quality.</p>
<p>Recommended citation: <b>Shamshad, M.</b>, Dhillon, S. K., &amp; kaur, G. (2016). "Heterosis for oil content and oil quality in sunflower (<i>Helianthus annuus</i> L.)". <i>Current Advances in Agricultural Sciences</i>(An International Journal), 8(1), 44. Retrieved from https://doi.org/10.5958/2394-4471.2016.00010.</p>
<!-- citation and icon code --> <a href="https://doi.org/%2010.5958/2394-4471.2016.00010.1"><i class="fas fa-fw fa-link zoom" aria-hidden="true"></i></a> <a href="/files/research/Heterosis_Oil_content.pdf"><i class="fas fa-fw fa-file-pdf zoom" aria-hidden="true"></i></a></article></div>
<div class="list__item"><article class="archive__item" itemscope="" itemtype="https://schema.org/CreativeWork"><h2 class="archive__item-title" itemprop="headline"> <a href="https://shamshad.in/publication/2016-01-12-Morphological-and-Genetic-diversity-in-sunflower" rel="permalink">Morphological and molecular genetic diversity analyses in <i>Helianthus annuus</i> (L.) </a>
</h2>
<p>Published in <i>Electronic Journal of Plant Breeding</i>, 2016</p>
<p class="archive__item-excerpt" itemprop="description"></p>
<p>The knowledge of genetic diversity among parental lines is a prerequisite for selecting parents for hybridization in any heterosis breeding programme. Evaluation of 13 parental lines using D2 analysis was undertaken during spring 2014 at PAU, Ludhiana. The genotypes were grouped into four clusters. Cluster II comprised of maximum number of genotypes (5) viz., CMS-47A, P 93R, P145R, P103R, P 167R followed by cluster I (4 genotypes i.e. CMS-11A, CMS-67A, CMS-234A, 95-C-1); cluster III and IV having two genotypes each i.e. CMS-68A, RCR 8297 and P 124R, P 134R respectively. Maximum inter cluster distance of 578.187 was recorded between genotypes of cluster II and IV while minimum inter cluster distance (175.195) was observed between genotypes present in cluster III and IV. Thirty SSR markers were used for molecular markers analysis. Total number of alleles amplified by 26 polymorphic primers was 51 with an average of 1.96 alleles per locus. The average value of polymorphic information content (PIC) for all the 26 polymorphic markers was 0.47. A polygenetic tree generated, based on Dice dissimilarity matrix and cluster analysis, led to the clustering of the sunflower genotypes into two major groups (one group comprising R lines while other comprising A lines) while one genotype (CMS 234A) formed separate independent cluster. The result of dissimilarity matrix revealed high genetic diversity among all the inbreds lines. Overall, the values for genetic distances ranged from 0.13 to 0.71. The highest genetic distance (0.71) was observed between the genotypes i.e. CMS 67A and P 93R followed by 0.70 for CMS 67A and P124R. Minimum genetic distance 0.13 was observed between P 103R and P 134R, P 134R and P 145R which was indicative of common parentage of these genotypes in the pedigree tree. The principle coordinated analysis along with pattern of clustering of dissimilarity matrix separated the lines into two groups; one having B-lines and other for R-lines. The analysis of molecular variance (AMOVA) showed greater variation within genotypes (68%) as compared to between genotypes (32%).</p>
<p>Recommended citation: <b>Shamshad, M.</b> &amp; Dhillon, S. K. (2016). "Morphological and molecular genetic diversity analyses in <i>Helianthus annuus</i> (L.)". <i>Electronic Journal of Plant Breeding</i>, 7(4), 1216-1223. Retrieved from https://doi.org/10.5958/0975-928X.2016.00169</p>
<!-- citation and icon code --> <a href="https://doi.org/10.5958/0975-928X.2016.00169.1"><i class="fas fa-fw fa-link zoom" aria-hidden="true"></i></a> <a href="/files/research/Morphological.pdf"><i class="fas fa-fw fa-file-pdf zoom" aria-hidden="true"></i></a></article></div>
<div class="list__item"><article class="archive__item" itemscope="" itemtype="https://schema.org/CreativeWork"><h2 class="archive__item-title" itemprop="headline"> <a href="https://shamshad.in/publication/2014-07-11-Combinig-Ability-sunflower" rel="permalink">Magnitude of combining ability for seed yield and oil content in sunflower (<i>Helianthus annuus </i>L.) </a>
</h2>
<p>Published in <i>National Symposium on Crop Improvement for Inclusive Sustainable Development</i>, 2014</p>
<p class="archive__item-excerpt" itemprop="description"></p>
<p>The additive components of genetic variance played the main role in the inheritance of oil content, which was confirmed by below one GCA/SCA ratio. The main role in the inheritance of seed yield was played by the non additive gene effects. The lines CMS 67A, CMS234A and P145R having highly significant gca effects for both seed yield and oil content should be used in future hybrid breeding programme to develop productive hybrids w.r.t. oil and seed yield</p>
<p>Recommended citation: <b>Shamshad, M.</b>, Dhillon, S. K. (2014). "Magnitude of combining ability for seed yield and oil content in sunflower (<i>Helianthus annuus</i> L.)". In <i>National Symposium on Crop Improvement for Inclusive Sustainable Development </i>(Vol. 10, pp. 248-250). Retrieved from https://doi.org/10.5958/0975-928X.2017.00014</p>
<!-- citation and icon code --> <a href="httpx//www:ripublication:com/%20ijafst:htm"><i class="fas fa-fw fa-link zoom" aria-hidden="true"></i></a> <a href="/files/research/Magnitude_combiningAblity_sunflower.pdf"><i class="fas fa-fw fa-file-pdf zoom" aria-hidden="true"></i></a></article></div>
<div class="list__item"><article class="archive__item" itemscope="" itemtype="https://schema.org/CreativeWork"><h2 class="archive__item-title" itemprop="headline"> <a href="https://shamshad.in/publication/2014-04-11-Genetic-Diversity-in-Sunflower" rel="permalink">Assessment of Genetic Diversity in Sunflower (<i>Helianthus annuus </i>L.) Germplasm </a>
</h2>
<p>Published in <i>International Journal of Agriculture and Food Science Technology</i>, 2014</p>
<p class="archive__item-excerpt" itemprop="description"></p>
<p>The present investigation was carried out at the research fields of the oilseeds Sections Department of Plant Breeding and Genetics Punjab Agricultural University, Ludhiana, India. The material for present study consisted of 31 germplasm lines. The material was raised in two rows with three replications each row of 4.5 m length with 60 cm and 30 cm inter and intra row spacing respectively, in the randomized block design. All the agronomic practices recommended for the region were followed to raise a good crop. The data for agronomic and yield traits i.e. plant heights, head diameters, volume weights, 100 seed weights, seed yield per plant was recorded. D2 analysis assigned the test genotypes into six clusters indicating presence of enough genetic diversity in the material. All the 31 germplasm lines were grouped into six clusters. The critical examination of clusters indicated the presence of high level of genetic diversity in the material. Cluster I comprised of maximum number of lines (18) and cluster III (9) cluster IIs IVs V and VI each consist one line respectively in normal environment. The genotypes included in the same cluster are considered genetically similar with respect to the aggregate effect of the characters examined the hybridization attempted between these is not expected to yield desirable recombinants. Therefore, putative parents for crossing programme should belong to different clusters characterized by large inter-cluster distance. The intra cluster distances ranged from 0 (cluster II, IV, V and VI) to 417.294 (cluster I) indicating presence of only one line in cluster II, IV, V and VI whereas, genotypes in cluster I were more dissimilar in agronomic and yield traits than other clusters. The maximum inter-cluster distance was observed between the members of cluster II with cluster VI (12836.64) followed by cluster I with cluster VI (7084.305) cluster V with cluster VI (6600.998) and cluster II with cluster IV (6498.283). The genotypes NC-41B (Morden), HOAL-38, P61R and TX-16R grouped in different clusters were identified as the most divers genotypes and should be used in hybridization program to realize high heterosis for these traits.</p>
<p>Recommended citation: <b>Shamshad, M.</b>, Dhillon, S. K., Tyagi, V., Akhatar, J. (2014). "Assessment of Genetic Diversity in Sunflower (<i>Helianthus annuus </i>L.) Germplasm". <i>International Journal of Agriculture and Food Science Technology</i>, 5(4), 267-272. Retrieved from httpx//www:ripublication:com/ ijafst:htm</p>
<!-- citation and icon code --> <a href="httpx//www:ripublication:com/%20ijafst:htm"><i class="fas fa-fw fa-link zoom" aria-hidden="true"></i></a> <a href="/files/research/Shamshad_ijafst.pdf"><i class="fas fa-fw fa-file-pdf zoom" aria-hidden="true"></i></a></article></div>
<!--<h2>Conference Papers</h2>--><h2>Books and Book chapters</h2>
<div class="list__item"><article class="archive__item" itemscope="" itemtype="https://schema.org/CreativeWork"><h2 class="archive__item-title" itemprop="headline"> <a href="https://shamshad.in/publication/2022-03-27-Tackling-a-Cereal-Killer-on-the-Run" rel="permalink">Tackling a Cereal Killer on the Run: Unending Fight Between Wheat Breeding and Foliar Rusts </a>
</h2>
<p>Published in <i>New Horizons in Wheat and Barley Research</i>, 2022</p>
<p class="archive__item-excerpt" itemprop="description"></p>
<p>Outbreaks of emerging new pathotypes of wheat rust pathogens, increasing at an alarming rate, are threatening the global food security. Wheat rusts caused by Puccinia species are major biotic constraints in efforts to sustain wheat production worldwide. Their quick evolution and capacity to spread over long distances make the resistance breeding in wheat a very challenging task. Pre-emptive or anticipatory breeding and sensible deployment of rust resistant cultivars have proven to be an effective strategy to manage wheat rusts. Efforts are focussed to accelerate rust resistance breeding strategies and explore wheat rust epidemiology. The collaborative role of wheat breeders and pathologists in addressing these threats to plant health is essential. This chapter presents the efforts done for rust resistance breeding at the global level and deployment of resistant cultivars in different geographical areas to combat the effect of stripe rust. Only marginal increase in wheat area is recorded, but the strategic deployment of rust resistance genes is most protective of crop production and crucial in sustaining the production levels of wheat.</p>
<p>Recommended citation: Sharma, A., *Shamshad, M.*, Kaur, S., Srivastava, P., Mavi, G.S., Sohu, V.S. (2022). Tackling a Cereal Killer on the Run: Unending Fight Between Wheat Breeding and Foliar Rusts. In: , et al. New Horizons in Wheat and Barley Research . Springer, Singapore. https://doi.org/10.1007/978-981-16-4449-8_10</p>
<!-- citation and icon code --> <a href="https://doi.org/10.1007/978-981-16-4449-8_10"><i class="fas fa-fw fa-link zoom" aria-hidden="true"></i></a> <a href="/files/research/bookChapter/tacklingCerealKiller.pdf"><i class="fas fa-fw fa-file-pdf zoom" aria-hidden="true"></i></a></article></div>
<div class="list__item"><article class="archive__item" itemscope="" itemtype="https://schema.org/CreativeWork"><h2 class="archive__item-title" itemprop="headline"> <a href="https://shamshad.in/publication/2018-10-30-Genomic-Selection" rel="permalink">The Usage of Genomic Selection Strategy in Plant Breeding </a>
</h2>
<p>Published in <i>Next Generation Plant Breeding, Intech Publisher</i>, 2018</p>
<p class="archive__item-excerpt" itemprop="description"></p>
<p>Major paradigm shift in plant breeding since the availability of molecular marker technology is that mapping and characterizing the genetic loci that control a trait will lead to improved breeding. Often, one of the rationales for cloning of QTL is to develop the “perfect marker” for MAS, perhaps based on a functional polymorphism. In contrast, an advantage of genomic selection is precisely its black box approach to exploiting genotyping technology to expedite genetic progress. This is an advantage in our view because it does not rely on a “breeding by design” engineering approach to cultivar development requiring knowledge of biological function before the creation of phenotypes. Breeders can therefore use genomic selection without the large upfront cost of obtaining that knowledge. In addition, genomic selection can maintain the creative nature of phenotypic selection which couple’s random mutation and recombination to sometimes arrive at solutions outside the engineer’s scope. Currently, the lion’s share of research on genomic selection has been performed in livestock breeding, where effective population size, extent of LD, breeding objectives, experimental design, and other characteristics of populations and breeding programs are quite different from those of crop species. Nevertheless, a great number of findings within this literature are very illuminating for genomic selection in crops and should be studied and built upon by crop geneticists and breeders. The application of powerful, relatively new statistical methods to the problem of high dimensional marker data for genomic selection has been nearly as important to the development of genomic selection as the creation of high-density marker platforms and greater computing power. The methods can be classified by what type of genetic architecture they try to capture.</p>
<p>Recommended citation: <b>Shamshad, M.</b>, Sharma, A. (2018). In: Yelda Ozden C (eds), The usage of genomic selection strategy in plant breeding, In eds. Next Generation Plant breeding, Intech Publisher 2018, Pages 93-108 DOI: 10.5772/intechopen.76247.</p>
<!-- citation and icon code --> <a href="https://www.intechopen.com/chapters/60786"><i class="fas fa-fw fa-link zoom" aria-hidden="true"></i></a> <a href="/files/research/bookChapter/Genomic-Selection-Strategy-Plant.pdf"><i class="fas fa-fw fa-file-pdf zoom" aria-hidden="true"></i></a></article></div>
<!--<h2>Academic</h2>--><hr>
<p><sup>*</sup> Equal authorship</p></section><footer class="page__meta"><p class="page__date"><strong><i class="fas fa-fw fa-calendar-alt" aria-hidden="true"></i> Updated:</strong> <time datetime="2024-10-06">October 6, 2024</time></p></footer>
</div></article>
</div>
</div><div id="footer" class="page__footer"><footer> <!-- start custom footer snippets --> <!-- end custom head snippets --> <!-- end custom footer snippets --><div class="page__footer-follow"><ul class="social-icons">
<li><strong>Follow:</strong></li>
<li><a href="https://twitter.com/DrShamshadM" rel="nofollow noopener noreferrer"><i class="fab fa-fw fa-twitter-square" aria-hidden="true"></i> Twitter</a></li>
<li><a href="https://scholar.google.com/citations?user=dI7V1a4AAAAJ&amp;hl=en" rel="nofollow noopener noreferrer"><i class="ai ai-google-scholar-square ai-fw" aria-hidden="true"></i> Google Scholar</a></li>
<li><a href="https://www.researchgate.net/profile/Mohd-Shamshad" rel="nofollow noopener noreferrer"><i class="ai ai-researchgate-square ai-fw" aria-hidden="true"></i> ResearchGate</a></li>
<li><a href="/feed.xml"><i class="fas fa-fw fa-rss-square" aria-hidden="true"></i> Feed</a></li>
</ul></div>
<div class="page__footer-copyright">© 2024 M. Shamshad. Proudly hosted by <a href="https://jekyllrb.com" rel="nofollow">Jekyll</a> &amp; <a href="https://mademistakes.com/work/minimal-mistakes-jekyll-theme/" rel="nofollow">Minimal Mistakes</a>.</div>
<link rel="preload" href="/assets/js/mermaid.min.js" defer as="script"> <script defer src="/assets/js/mermaid.min.js"></script></footer></div><link async rel="preload" href="/assets/js/main.min.js" as="script"> <script defer src="/assets/js/main.min.js"></script> <!-- <script src="https://kit.fontawesome.com/4eee35f757.js"></script> --></body></html>
